# Properties of epigenome-wide association studies {#properties-of-ewas}

```{r call_source-04}
read_chunk("chunks/04-properties_of_ewas.R")
```

``` {r load-data-04, results="hide", message=FALSE}
```

``` {r study-data-setup, results="hide", message=FALSE}
```

``` {r traits-manhattan-setup, results = "hide", message=FALSE}
```

``` {r rsq-setup, results = "hide", message=FALSE}
```

``` {r batch-faulty-setup, results = "hide", message=FALSE}
```

``` {r replication-setup, results = "hide", message=FALSE}
```

``` {r cpg-characteristics-setup, results = "hide", message=FALSE}
```

``` {r enrichment-setup, results = "hide", message=FALSE}
```

## Chapter summary {#chapter-summary-04}
The EWAS Catalog database developed in __Chapter \@ref(ewas-catalog)__ contains thousands of associations from hundreds of studies. By far the most common method of measuring DNA methylation amongst these EWAS is in blood using the Illumina Infinium HumanMethylation450 BeadChip (HM450 array). This platform assays fewer than 2% of CpG sites in the human genome, and those selected are ascertained for regions hypothesised to be relevant to gene regulation. Understanding what drives the associations found by measuring DNA methylation in this way could help prioritise CpG sites or regions of the genome to target for future technologies used in EWAS and further, it could guide current EWAS study design (for example by discovering sites which could be removed before analysis). 

In this chapter I use the data collected for The EWAS Catalog to evaluate the characteristics of known DNA methylation associations. Of the data in the EWAS Catalog, `r comma(percent_faulty_probes + percent_sex_probes)`% of reported associations are from CpGs measured by probes recently reported as unreliable and `r comma(100 - (sum(batch_tab$batch_included & batch_tab$cell_comp_included) / nrow(batch_tab) * 100))`% of studies did not account for both batch effects and cellular composition, suggesting some associations may be false positives. However, after removal of these potentially false positive results, characteristics of DNA methylation still associated with EWAS results. An increase in heritability by 10% associated with a decrease in the modulus of the standardised effect size by `r comma((1 - get_cc_res("effect", "h2")[["Beta"]]) * 100)`% [95% CI: `r comma((1 - get_cc_res("effect", "h2")[["2.5 %"]]) * 100)`% decrease, `r comma((1 - get_cc_res("effect", "h2")[["97.5 %"]]) * 100)`% decrease]. It was also observed that the modulus of the standardised effect size decreased by `r comma((1 - get_cc_res("effect", "varcpg_per_sd")[["Beta"]]) * 100)`% [95% CI: `r comma((1 - get_cc_res("effect", "varcpg_per_sd")[["2.5 %"]]) * 100)`% decrease, `r comma((1 - get_cc_res("effect", "varcpg_per_sd")[["97.5 %"]]) * 100)`% decrease] per one standard deviation increase in the variance of DNA methylation. Differentially methylated positions (DMPs) were found to be present in actively transcribed promoter regions, enhancer regions and in over 100 transcription factor binding sites more than expected by chance. DMPs were also found to be depleted for heterochromatic regions, poised and bivalent promoters, regions repressed by polycomb proteins and quiescent regions. Targeting genomic regions for measurement of DNA methylation that are enriched for EWAS results, and avoiding regions that are depleted for EWAS results, may increase probability of identifying DMPs in future EWAS. These enrichment and depletion analyses were conducted across 25 distinct tissues and the strongest associations were found in blood, suggesting conducting EWAS in different tissues will likely yield tissue-specific results.

The results presented in this chapter (and later chapters) are using data from The EWAS Catalog when published data present in the database was from before 2019-01-01.

### Contributions statement {#contributions-statement-04}

I wrote all the sections and performed all analyses except the the enrichment analyses using LOLA, which was performed by Charlie Hatcher. I performed the downstream analyses from LOLA output (visualisation of the results and some further statistical analyses).

## Introduction {#introduction-04}
Hundreds of EWAS have been conducted in the last 10-15 years, yet no systematic evaluation of published EWAS across complex traits has been conducted. By exploring the patterns of association across a large group of EWAS, one can discover potential explanations for the results found, that may shed light on technical issues affecting previous studies as well as shared epigenetic architectures across traits.

Since the inception of EWAS, it has become clear that batch effects and cellular heterogeneity can generate false positives and bias effect sizes [@Price2018; @Forest2018; @Jaffe2014]. Also, as discussed in __Section \@ref(measuring-dna-methylation)__, characterisation of probes used by common arrays (e.g. the HM450 array and the HMEPIC array) has shown that unreliable methylation measurements may occur because of cross-hybridisation of probes, non-specific probe mapping and SNPs being present at the binding sites of probes [@Zhou2017; @Naeem2014]. Despite this, there are examples of replication amongst EWAS results, [@Nano2017; @Kaushal2017; @Morris2017; @Hedman2017; @Braun2017; @Teschendorff2015; @Zeilinger2013]. Further functional characterisation of EWAS results, such as new experimental studies or the application of existing gene function knowledge, can also be used to bolster evidence that changes in DNA methylation estimated are unlikely due to bias [@Rakyan2011; @Corbin2019]. By way of an example, changes in DNA methylation at _AHRR_ have been replicated across multiple smoking EWAS [@Zeilinger2013; @Elliott2014; @Joehanes2016; @Bojesen2017] and as functional research has implicated this gene in handling toxic substances found in tobacco smoke [@Zudaire2008], it seems unlikely these findings are chance occurrences. 

The characteristics of the DNA methylome may also explain some EWAS findings. Heritability varies across DNA methylation sites [VanDongen2016; @McRae2014; @Hannon2018], and so if genetic effects are driving EWAS associations, either through confounding or with DNA methylation as a mediator, one would expect heritable sites to be commonly identified in EWAS. Variance is also heterogenous across sites [@Garg2018] and at sites where variation is low, the ratio of noise to signal may be greater. Thus, some studies have advocated removing these sites to prevent generating false positives and to reduce the multiple testing burden [@Meng2010; @Logue2017]. However, it is unclear how variance in DNA methylation relates to the magnitude of effect estimates. 

Experimental studies have shown DNA methylation changes at different locations of the genome correlate with different regulatory functions. For example, an increase in DNA methylation at transcriptional start sites is correlated with a decrease in gene expression [@Jones2012; @Ando2019; @Deaton2011], but an increase in DNA methylation within a gene body shows the opposite association [@Wolf1984; @Hellman2007]. As discussed in __Section \@ref(ewas)__, the understanding that the genomic contexts in which DNA methylation occurs is related to gene regulation likely contributed to the design of contemporary arrays that measure DNA methylation. Yet, it is not known whether targeting protein-coding regions and enhancers has likely led to an increase in discovery of DNA methylation-trait associations.

Understanding underlying factors that drive EWAS results is essential for future study design. This may come in the form of consideration of potential biasing factors, or by selecting certain DNA methylation sites based on their specific characteristics. Further, the HM450 and HMEPIC arrays both capture less than 5% of the total number of CpG sites in the genome, therefore understanding the characteristics of DNA methylation-trait associations could provide vital information when designing future studies targeting the other 95%.

Also, by examining the commonalities of EWAS results, one has the potential to uncover links between traits that have not previously been made or to identify new potential mediating factors between traits.

In this chapter I first describe the data present in The EWAS Catalog going on to explore the factors that predict EWAS hits.

\newpage

## Methods {#methods-04}

### Epigenome-wide association studies data {#ewas-data-04}
All the data for the analyses were extracted from The EWAS Catalog (__Chapter \@ref(ewas-catalog)__). Data were extracted when The EWAS Catalog had published EWAS data from before the start of 2019 (i.e. the data do not completely reflect that presented in __Chapter \@ref(ewas-catalog)__). Studies were removed that compared DNA methylation levels between tissue, race, and age. This was done because these variables are not complex traits and thus the properties of those study results are unlikely to be informative when attempting to understand how to best design EWAS. Overall, this left `r n_ewas_04` EWAS, including `r n_aries` EWAS from the ARIES subsection of ALSPAC (__Section \@ref(aries-02)__) [@Relton2015-aries; @Fraser2013; @Boyd2013] and 40 EWAS performed using data from the gene expression omnibus (GEO) resource (__Section \@ref(geo-02)__). See __Chapter \@ref(ewas-catalog)__ for more details on the EWAS.

### Description of catalog data {-#description-of-data}
DMPs, unless otherwise stated, will be defined as DNA methylation sites associated with a trait at P<1x10^-7^. Each of the CpGs in the Catalog are annotated to genes, using data from the meffil R package [@Min2018]. 

T-statistics ($t$) were calculated using P-values, sample sizes ($n$) and the qt() function in R. $r^2$ values were calculated from t-statistics as follows

\begin{equation}
    r^2 = \frac{t^2} {t^2 + n - 1}
    (\#eq:r-squared)
\end{equation}

To identify which traits for which r^2^ values might be inflated, r^2^ values were first summed and divided by the sample size (N). The base 10 log of these values were taken to approximate a normal distribution. Then a z-test was performed to assess for which studies the sum of r^2^ values divided by N were greater than the mean of summed r^2^ values divided by N across all studies. From the z-test, those with a FDR-corrected P-value of less than 0.05 were labelled as having inflated r^2^ values. 

### Identifying faulty probes {#identifying-faulty-probes}
By far the most common method to measure DNA methylation across the studies in The EWAS Catalog is using the HM450 array. Since its development, the array has been extensively characterised [@Price2018; @Forest2018; @Jaffe2014; @Zhou2017] and it was found not all probes measure DNA methylation reliably. Some probes map to CpG sites that are influenced by SNPs, others are non-specific and some are prone to cross-hybridisation. Probes were assigned to be 'potentially faulty' if they were characterised as such by Zhou et al. [@Zhou2017]. 

### Replication {#replication-methods-04}
A study-wide significant association (P<1x10^-7^) was deemed to be replicated if it had been identified by another study at P<1x10^-4^ (i.e. it was replicated if reported by another study in The EWAS Catalog). It should be re-iterated that the published data collected for The EWAS Catalog was scraped from the journal articles and even though the threshold for inclusion into The EWAS Catalog remained constant at P<1x10^-4^, there is heterogeneity in reporting of EWAS associations. Therefore, some studies would not have reported any results with a P-value lower than the conventional EWAS P-value threshold (P<1x10^-7^), making power a key limitation for attempts to assess replication. The replicability of EWAS within the database was assessed using two methods. Firstly, replication within studies is recorded in the EWAS Catalog, thus a simple lookup for any studies that performed a replication or meta-analysed discovery and replication datasets was conducted. Secondly, a lookup of results for any traits for which multiple EWAS had been conducted was performed. 

The Catalog also contains results from studies that have uploaded their data to GEO as well as results from the re-analysis of that data (details in __Section \@ref(new-ewas-03)__). These re-analyses adjusted for 20 surrogate variables only as many studies did not provide a complete set of covariates to GEO. To assess the potential reliability of these EWAS, the original EWAS results were looked up in the results of the re-analysed data. 

### Selecting data to assess DNA methylation characteristics
Before further analyses, all potentially faulty probes and probes that mapped to sex chromosomes were removed. Studies that did not include batch and cell composition as covariates in at least one EWAS model were also excluded, and studies for which re-analysis of the data replicated less than 10% of the findings were removed.

### DNA methylation characteristics
The relationship between the characteristics, heritability and variance, of DNA methylation at each CpG site and EWAS effect size was assessed. To allow this across traits, beta coefficients were standardised, $\beta_{standard}$, like so, 

\begin{equation}
    \beta_{standard} = \frac{\beta\sigma(x)} {\sigma(y)}
    (\#eq:standardised-beta-coeffs)
\end{equation}

where $\beta$ = beta coefficient, $\sigma$ = standard deviation, $x$ = independent variable, $y$ = dependent variable. As individual participant data were not available to us, the variance in DNA methylation sites was approximated by the variance in DNA methylation at sites as supplied by the Genetics of DNA Methylation Consortium (GoDMC) [@Min2020] and the trait variance was estimated by rearranging equation \@ref(eq:r-squared-from-beta) depending on whether DNA methylation was the independent ($x$) or dependent ($y$) variable in the model.

\begin{equation}
	r^2 = \frac{\beta^2\sigma^2(x)} {\sigma^2(y)}
	(\#eq:r-squared-from-beta)
\end{equation}

GoDMC [@Min2020] also provided the mean levels of DNA methylation at each site. Heritability of DNA methylation at each site has been previously estimated by McRae et al. 2014 [@McRae2014] and Van Dongen et al. 2016 [@VanDongen2016]. These values were kindly made publically available by the authors of those studies, in this chapter the estimates of heritability from twin data (Van Dongen et al. 2016 [@VanDongen2016]) were used.

Relationships between each characteristic and effect size were assessed using linear regression, fitting the absolute value of the standardised effect size as the dependent variable and the characteristic as the independent variable. The absolute values of standardised effect sizes were transformed using the natural log to approximate normality.

The relationship between tendency for a DMP to replicate and heritability and variance were also assessed. Logistic regression models were fitted with the binary variable of a DMP replicating in at least one study (yes or no) used as the outcome measure and heritability and variance fitted as the dependent variable.

It was also tested whether heritability and variance could predict whether a CpG site was likely to be identified as a DMP in EWAS by generating receiver operating characteristic (ROC) curves and quantifying the area under these curves (AUC).

### Enrichment tests {#enrichment-tests-04}
Assessment of enrichment of DMPs amongst various genomic regions was carried out to help understand whether selecting regions to measure could maximise EWAS yield. Locus Overlap Analysis (LOLA) [@Sheffield2016] was used to assess whether DMPs identified in the EWAS Catalog were enriched for 25 chromatin states and 167 transcription factor binding sites in 127 different cell types comprising 25 distinct tissues. These data were generated by the Roadmap Epigenomics Project [@Kundaje2015] and ENCODE [@Dunham2012].

Five different groups of DMPs were defined for the enrichment analyses:

* Group A - all sites associated with any complex trait at the conventional P-value threshold used in EWAS, P<1x10^-7^.
* Group B - a subset of group A, all sites associated with any complex trait at a more stringent threshold, P<`r strict_p_threshold`. Multiple EWAS were conducted to produce the results in the database and so the stricter threshold of group B aimed to limit the false discovery rate by taking into account the multiple EWAS.
* Group C - DMPs replicated at P<1x10^-4^ in any other EWAS of the same trait.
* Group D - a subset of group A, but restricted to results from studies where DNA methylation was measured in whole blood.
* Group E - a subset of group B, but restricted to results from studies where DNA methylation was measured in whole blood.

To assess enrichment, LOLA performs Fisher's exact test and generates an odds ratio that can be interpreted as the odds of the DMPs being within an annotation divided by the odds of the DMPs not being within an annotation. Genomic annotations may differ by CG content and thus a differential CG content of regions containing the DMPs of interest and the background group of CpG sites might bias enrichment estimates. Thus, background sites were matched on CG content before the analysis.


All analyses were completed using R (version 3.6.2).

\newpage

## Results {#results-04}

### Description of the catalog {-#catalog-description}
Before assessing the factors predicting DMPs, a brief summary of the data in The EWAS Catalog (as used for this study, see __Section \@ref(ewas-data-04)__ for details) is presented (__Table \@ref(tab:study-data-tab)__).

The percentage of hypermethylated sites in relation to traits was `r comma(dir_of_effect$hyper * 100)`% and there were `r num_to_text(nrow(h_genes))` CpGs that associated with more than ten traits (__Figure \@ref(fig:traits-manhattan)__). Here are those sites with gene names, as mapped using Illumina-provided annotations, in brackets: `r h_cpg_genes`. `r h_cpgs[length(h_cpgs)]` (_`r names(h_cpgs)[length(h_cpgs)]`_) was associated with more traits than any other site - `r max(h_genes$n_traits)` traits. These correspond mostly to metabolites, weight-related traits, and type two diabetes. 

```{r study-data-tab, message=FALSE, results="asis"}
```

Next estimates of the trait variance (see equation \@ref(eq:r-squared)) explained by each association were assessed. This indicates the predictive performance from EWAS, although it should be noted that winner's curse will artificially inflate the performance, even amongst EWAS with true positive results.

The proportion of trait variance (ranging from 0 to 1) that correlated with DNA methylation (r^2^) at each site varied from `r comma(min_r2)` to `r comma(max_r2)` with a median of `r comma(median(rsq_dat$rsq, na.rm = T))` (__Figure \@ref(fig:rsq-distribution)__). The sum of r^2^ values ranged greatly from `r comma(min(rsq_sum_dat$total_rsq))` to `r comma(max(rsq_sum_dat$total_rsq))` (__Figure \@ref(fig:rsq-sum-distribution)__), with a median of `r comma(median(rsq_sum_dat$total_rsq))`. There was evidence that `r num_to_text(nrow(suspect_studies))` studies had a total sum of r^2^ values greater than the mean (FDR < 0.05) and r^2^ values from individual associations from these studies made up the majority of r^2^ values greater than 0.1 (__Figure \@ref(fig:rsq-distribution)__). When excluding those studies from the results, the median r^2^ value at individual sites was `r comma(new_med_rsq)`.

(ref:rsq-distribution-cap) __Distribution of r^2^ values across all CpG sites in The EWAS Catalog__. Each EWAS can identify multiple differentially methylated positions, each of which will capture some variance of the trait of interest for that EWAS (r^2^). $\sum {r^2}$ is the sum of r^2^ values, the distribution of which is shown in __Figure \@ref(fig:rsq-sum-distribution)__. `r num_to_text(nrow(suspect_studies), start_of_sentence = TRUE)` studies were identified for which there was some evidence that the sum of r^2^ values were greater than the mean across all studies. All of the differentially methylated positions identified by those studies are highlighted in blue on the plot.

These results suggest that some associations within the database are likely to be inflated, yet for most traits, variation at individual DNA methylation sites captures little trait variance. Summing the r^2^ values indicates a substantial proportion of trait variance can be captured by multiple DNA methylation sites for some traits, but this can only be estimated by jointly modelling the contribution of all sites to trait variance. This is explored in __Chapter \@ref(h2ewas-chapter)__. Here, the sum of r^2^ values is used to indicate whether the results of a study are likely inflated and thus may not be reliable. 

\pagebreak

(ref:traits-manhattan-cap) __Number of unique traits associated with DNA methylation at each CpG__. Sites associated with more than 10 unique traits are highlighted in orange and labelled.

(ref:traits-manhattan-scap) Number of unique traits associated with DNA methylation at each CpG

```{r traits-manhattan, message=FALSE, results="asis", fig.cap='(ref:traits-manhattan-cap)', fig.pos="!hp", fig.scap='(ref:traits-manhattan-scap)'}
```

(ref:rsq-distribution-scap) Distribution of r^2^ values across all CpG sites in The EWAS Catalog

```{r rsq-distribution, message=FALSE, results="asis", fig.cap='(ref:rsq-distribution-cap)', fig.pos="!hp", fig.scap='(ref:rsq-distribution-scap)'}
```

(ref:rsq-sum-distribution-cap) __Distribution of the sum of r^2^ values across each study in The EWAS Catalog__.

(ref:rsq-sum-distribution-scap) Distribution of the sum of r^2^ values across each study in The EWAS Catalog

```{r rsq-sum-distribution, message=FALSE, results="asis", fig.cap='(ref:rsq-sum-distribution-cap)', fig.pos="!hp", fig.scap='(ref:rsq-sum-distribution-scap)'}
```

\pagebreak

### Robustness of results {#robustness-of-results}
As discussed, cellular heterogeneity, batch effects and inclusion of faulty probes can lead to false positives in EWAS. The extent to which this might be the case within EWAS included within The EWAS Catalog was explored.

Each study may have reported results across multiple EWAS models, adjusting for different covariates. In at least one model, `r sum(batch_tab$batch_included)` studies adjusted for batch effects, `r sum(batch_tab$cell_comp_included)` studies adjusted for cell composition, and `r sum(batch_tab$batch_included & batch_tab$cell_comp_included)` adjusted for both. Of all DMPs identified, `r comma(percent_faulty_probes)`% were measured by potentially faulty probes and an extra `r comma(percent_sex_probes)`% were present on sex chromosomes (__Figure \@ref(fig:faulty-probes-plot)__). 

(ref:faulty-probes-cap) __The percentage of DMPs that may have been identified by faulty probes and the percentage of EWAS that reported identifying at least one of these probes__. The left-hand bar represents all DMPs reported across all EWAS that fit into the categories shown, the right-hand bar represents the number of EWAS that include CpGs that fit into the categories shown. Some CpGs are both on a sex chromosome and were identified as faulty by Zhou et al. They were labelled as 'potentially faulty'.

(ref:faulty-probes-scap) The percentage of DMPs that may have been identified by faulty probes and the percentage of EWAS that reported identifying at least one of these probes

```{r faulty-probes-plot, message=FALSE, results="asis", fig.cap='(ref:faulty-probes-cap)', fig.pos="!h", fig.scap='(ref:faulty-probes-scap)'}
```

There were `r rep_study_n[rep_study_n$study == "discovery_and_replication", "n", drop=T]` studies that performed a meta-analysis of discovery and replication samples. A further `r rep_study_n[rep_study_n$study == "replication", "n", drop=T]` studies performed a separate replication analysis. Together, this provides `r nrow(replication_dat$rep_assoc)` associations within the EWAS Catalog that have been replicated at P < 1x10^-4^. 

From the studies that uploaded their data to GEO, the association between DNA methylation and the phenotype of interest from the original study was re-analysed, including 20 surrogate variables as covariates. Both the original study results and the results from the re-analysis of the phenotype of interest are in The EWAS Catalog database for `r nrow(geo_rean)` studies. Across the studies, between `r min(geo_rean$rep_percent)`% and `r max(geo_rean$rep_percent)`% of DMPs were replicated at P < 1x10^-4^ (__Table \@ref(tab:geo-reanalysis-tab)__). Some of these EWAS reported very few DMPs (some only 1) and as they would have used different models, replicating the single reported result was not expected. 

```{r geo-reanalysis-tab, message=FALSE, results="asis"}
```

Using the catalog data, DMPs were examined to see if they were also associated with that same trait in another study at P<1x10^-4^. There were `r comma(nrow(rep_rates))` studies that shared a common phenotype of interest. Replication rate, judged as the percentage of CpGs also present in any other study of the same trait with P<1x10^-4^, varied from `r comma(min(rep_rates$prop_rep)*100)` to `r comma(max(rep_rates$prop_rep)*100)` between studies (__Table \@ref(tab:replication-tab)__, __Table \@ref(tab:replication-tab-smoking)__, __Table \@ref(tab:replication-tab-bmi)__). For many of the traits, the number of identified DMPs was low (`r sum(rep_rates$N_DMPs == 1)` studies reported one DMP), therefore the low replication rate for these studies is not completely unexpected given potential study heterogeneity in important factors such as study power, age, sex, ancestry and study design. However, there were also three studies that identified over 100 DMPs and none of them replicated, including an EWAS of smoking for which there are many high-powered replication studies. \linebreak

```{r replication-tab, message=FALSE, results="asis"}
```

```{r replication-tab-smoking, message=FALSE, results="asis"}
```

```{r replication-tab-bmi, message=FALSE, results="asis"}
```

\pagebreak

Before continuing to assess what CpG characteristics might, in part, explain some associations found in EWAS, sites were removed that were identified by potentially faulty probes and were on either of the sex chromosomes. Further, studies that did not include batch effects and cell composition as covariates in at least one EWAS model were removed and studies for which fewer than 10% of sites identified in the original analyses were identified in a re-analysis using the data provided via GEO. Overall, this left `r rob_summary$n_ewas` EWAS and `r rob_summary$n_res` associations (at P<1x10^-4^).


<!-- Smoking is associated with large changes in DNA methylation across the genome (REF) and is associated with many different traits (REF). Thus, it may confound DNA methylation associations found in the catalog. If this was the case, one might expect smoking related CpGs to appear more in the catalog than expected by chance. The DMPs identified by EWAS of traits other than smoking were enriched for smoking related CpG sites (P = X).
 -->
<!-- ### Correlation across tissues {-#correlation-across-tissues}

* There is correlation between DNA methylation sites across tissues, suggesting stability in DNAm, but would we expect to find associations at these positions across tissues?

* Replication of sites across tissues for same traits? 
* Do sites that are highly correlated across tissues appear more than expected by chance in whole blood EWAS? 
  + If yes then suggests the correlation might be due to decreased measurement error at those sites
  + If the opposite (correlated sites appear less than expected by chance), then it suggests that correlation occurs at positions that don't really matter (i.e. are just stable because of things like being at housekeeping genes)
* Can check housekeeping gene theory if needed
 -->

### CpG characteristics {#cpg-characteristics}

Using the selected EWAS results, it was investigated whether the characteristics of DNA methylation at CpG sites explained associations found in EWAS. 

It has previously been suggested that sites at which DNA methylation variability is low should be removed [@Meng2010; @Logue2017]. The rationale for this is that if total variation is low then the ratio of variation due to technical effects to variation due to biological effects will be greater and thus any association with a complex trait is more likely to be due to technical artefacts. However, it's unknown whether this may be removing sites pertinent to complex trait variation.

When assessing whether variance at a CpG site was associated with the odds of a DMP being replicated in at least one other study, it was found that an increase of variance by one standard deviation associated with a decreased odds of a DMP replicating (OR = `r comma(get_cc_res("rep", "varcpg_per_sd")[["OR"]])` [95% CI: `r comma(get_cc_res("rep", "varcpg_per_sd")[["2.5 %"]])`, `r comma(get_cc_res("rep", "varcpg_per_sd")[["97.5 %"]])`] per sd increase in CpG variance). This contrasts with the notion that removing sites with lower variance will reduce the chances of identifying sites due to technical artefacts. 

There was also strong evidence of an inverse association between variance at a CpG site and effect size. It was observed that an increase in the variance of DNA methylation by one standard deviation was associated with a decrease in absolute standardised effect size by `r comma((1 - get_cc_res("effect", "varcpg_per_sd")[["Beta"]]) * 100)`% [95% CI: `r comma((1 - get_cc_res("effect", "varcpg_per_sd")[["2.5 %"]]) * 100)`% decrease, `r comma((1 - get_cc_res("effect", "varcpg_per_sd")[["97.5 %"]]) * 100)`% decrease]. This suggests that removal of sites with little variation may reduce the chances of discovering changes in DNA methylation that have larger effects, whilst not removing more unreliable sites. 

<!-- DNA methylation is a binary feature, a CpG site can either be methylated or not on a particular DNA molecule. However, when measuring methylation across multiple DNA molecules, the proportion of those molecules methylated at a given site will be between 0 and 1. If DNA methylation at a given site is important for specific regulatory functions within a group of cells, one might expect that site to be methylated (or unmethylated) in the majority of the cells. Thus, changes in methylation away from an extreme, might have more of an impact on cellular function.

There was strong evidence of an association between mean DNA methylation levels and negative effect sizes (P = `r comma(char_vals[['avg-meth (beta<0)']]$p)`, __Table \@ref(tab:cpg-chars-tab)__) and an inverse association between mean methylation levels and positive effect sizes (P = `r comma(char_vals[['avg-meth (beta>0)']]$p)`, __Table \@ref(tab:cpg-chars-tab)__).
 -->

DNA methylation changes are heritable [@McRae2014; @VanDongen2016], and DNA methylation could mediate the effects of genotype on complex traits or genotype might confound the association between DNA methylation and complex traits.  

It was found that an increase in heritability by 10% was associated with an increased odds of a DMP replicating (OR = `r comma(get_cc_res("rep", "h2")[["OR"]])` [95% CI: `r comma(get_cc_res("rep", "h2")[["2.5 %"]])`, `r comma(get_cc_res("rep", "h2")[["97.5 %"]])`]). An increase in heritability by 10% also associated with a decrease in absolute standardised effect size by `r comma((1 - get_cc_res("effect", "h2")[["Beta"]]) * 100)`% [95% CI: `r comma((1 - get_cc_res("effect", "h2")[["2.5 %"]]) * 100)`% decrease, `r comma((1 - get_cc_res("effect", "h2")[["97.5 %"]]) * 100)`% decrease].

Heritability also had a capacity to predict whether a site would be identified as a DMP in at least one study (AUC = `r comma(get_cc_res("dmp", "h2")[["auc"]])`), whereas variability only had a modest ability to do so (AUC = `r comma(get_cc_res("dmp", "variance")[["auc"]])`) and further did not add to the predictive capacity of heritability (combined AUC = `r comma(get_cc_res("dmp", "h2_and_variance")[["auc"]])`).

<!-- There was evidence that effect sizes tended to be greater in more heritable sites (P = `r comma(char_vals[['h2']]$p)`, __Table \@ref(tab:cpg-chars-tab)__). 

The combined variance in effect size estimates explained by DNA methylation variability and heritability was `r comma(char_vals[['variance + h2']]$r2)` (__Table \@ref(tab:cpg-chars-tab)__). \linebreak

```{r cpg-chars-tab, message=FALSE, results="asis"}
```

\linebreak
 -->

### Enrichment of DMPs for genomic annotations {#dmp-enrichment-results}

As the position of DNA methylation relative to genes is pertinent to its association with gene expression (__Section \@ref(dna-methylation)__) [@Jones2012; @Ando2019; @Deaton2011; @Wolf1984; @Hellman2007], the enrichment of DMPs identified in The EWAS Catalog across genomic regions and chromatin states were assessed (__Figure \@ref(fig:chrom-state-plot)__). Across all tissues, there was a trend for sites to be enriched for promoter regions (OR > 1). Evidence of enrichment across different enhancer types was mixed and there was a trend towards depletion of sites within heterochromatic regions, poised and bivalent promoters, regions repressed by polycomb proteins and quiescent regions (__Figure \@ref(fig:chrom-state-plot)__, OR < 1). 

(ref:chrom-state-cap) __Enrichment of DMPs for 25 chromatin states__. Chromatin states across the genome of 127 cell types comprising 25 distinct tissues were available from the Roadmap Epigenomics Project. Using LOLA, the enrichment of DMPs from across all data in The EWAS Catalog for chromatin states were assessed. DMPs were divided into five categories as detailed in __Section \@ref(enrichment-tests-04)__. The x-axis show the 25 chromatin states: TssA, Active TSS; PromU, Promoter Upstream TSS; PromD1, Promoter Downstream TSS with DNase; PromD2, Promoter Downstream TSS; Tx5', Transcription 5'; Tx, Transcription; Tx3', Transcription 3'; TxWk, Weak transcription; TxReg, Transcription Regulatory; TxEnh5', Transcription 5' Enhancer; TxEnh3', Transcription 3' Enhancer; TxEnhW, Transcription Weak Enhancer; EnhA1, Active Enhancer 1; EnhA2, Active Enhancer 2; EnhAF, Active Enhancer Flank; EnhW1, Weak Enhancer 1; EnhW2, Weak Enhancer 2; EnhAc, Enhancer Acetylation Only; DNase, DNase only; ZNF/Rpts, ZNF genes & repeats; Het, Heterochromatin; PromP, Poised Promoter; PromBiv, Bivalent Promoter; ReprPC, Repressed PolyComb, Quies, Quiescent/Low.

(ref:chrom-state-scap) Enrichment of DMPs for 25 chromatin states

```{r chrom-state-plot, message=FALSE, results="asis", fig.cap='(ref:chrom-state-cap)', fig.pos="!htp", fig.scap='(ref:chrom-state-scap)'}
```

The DMPs identified by EWAS were also enriched for transcription factor binding sites. Of the 167 transcription factor binding sites tested, there was evidence that identified DMPs  were enriched in `r tfbs_en$n_enrich` of them in at least one tissue type (FDR < 0.05). The strongest enrichments were found for transcription factor binding sites as measured in blood (median OR ranged from `r comma(min(tfbs_en$blood_enrich_or$median_or))` to `r comma(max(tfbs_en$blood_enrich_or$median_or))` based on how DMPs were defined - see __Section \@ref(enrichment-tests-04)__ for details) and varied from tissue to tissue, but overall enrichment was observed more often than not across all tissues (__Figure \@ref(fig:tfbs-plot)__). 

(ref:tfbs-cap) __Enrichment of DMPs for 167 transcription factor binding sites__. Using LOLA, the enrichment of DMPs from across all data in The EWAS Catalog for 167 transcription factor binding sites confirmed across 25 distinct tissues were assessed. DMPs were divided into five categories as detailed in __Section \@ref(enrichment-tests-04)__. The x-axis show the 25 distinct tissues. All transcription factor binding sites have not been confirmed across all tissues. For some tissues (e.g. "Eye" and "Gingiva") only five have been confirmed, but in blood over 131 have been confirmed.

(ref:tfbs-scap) Enrichment of DMPs for 167 transcription factor binding sites

```{r tfbs-plot, message=FALSE, results="asis", fig.cap='(ref:tfbs-cap)', fig.scap='(ref:tfbs-scap)'}
```

\newpage

## Discussion {#discussion-04}

Understanding the nature of EWAS associations is imperative for biological inference. Using data from The EWAS Catalog this chapter shows that many CpGs associate with multiple different unique traits and the magnitude of these associations are partly explained by the characteristics of DNA methylation levels. False positives may also explain a proportion of EWAS associations. Roughly 10% of the DMPs identified were measured by potentially faulty probes and the median percentage of CpGs that could be replicated across studies was `r comma(median(rep_rates$prop_rep) * 100)`%.

### Identifying mediators {#identifying-mediators}

Identifying modifiable molecular traits that mediate the effect of complex traits on disease is something that motivates a substantial portion of molecular epidemiology research [@Relton2010; @Relton2012]. Having a database of associations between DNA methylation and various traits and diseases may enable easy identification of potential mediators that warrant follow-up. Overall, DNA methylation at `r comma(sum(t_dat$n_traits > 0))` CpGs are associated with multiple traits. The CpG that was identified in the most EWAS, `r h_cpgs[length(h_cpgs)]` (_`r names(h_cpgs)[length(h_cpgs)]`_), had evidence from multiple studies that methylation at that site associated with weight-related traits such as body mass index [@Mendelson2017; @Wahl2017; @Shah2015; @Demerath2015] and waist circumference [@Demerath2015], roughly 60 metabolites [@Hedman2017; @Sayols-Baixeras2016; @Braun2017; @Kriebel2016; @Pfeiffer2015] and with type 2 diabetes [@Chambers2015]. Some studies have explored these associations further, for example, two studies used Mendelian randomization (MR) to provide evidence that body mass index caused changes in methylation at this site [@Mendelson2017; @Wahl2017]. However, full characterisation and assessment of whether methylation at that site mediates the effect of adverse adiposity on any diseases has not been undertaken and could be followed-up. 

### Biased results {#biased-results}

The potential biases in EWAS have been well documented [@Birney2016] and were discussed at length in __Section \@ref(problems-for-ewas)__. It is encouraging that the majority of studies include batch effects and cell composition in at least one of their models (`r comma(sum(batch_tab$batch_included & batch_tab$cell_comp_included) / nrow(batch_tab) * 100)`%). However, there are still some studies including probes that have been characterised as faulty.

Differences in cell composition, sample ethnicity, covariates used and other differential biases between studies might explain the low replication rate in some cases. However, studies only tend to report associations below the conventional EWAS P-value threshold, P<1x10^-7^, so differences in study power could also be a major factor.

### Understanding CpG characteristics {#understanding-cpg-characteristics}

Characteristics of DNA methylation discovered in experimental studies, such as its association with gene expression, were used to select sites to measure DNA methylation for array-based technology used in many EWAS [@Illumina2012]. Individual studies may also select CpGs to analyse from those assayed based on statistical characteristics such as high variance [@Meng2010; @Logue2017]. 

Our results suggest removing CpG sites with low variances may make it more likely to remove sites with greater effects. Variance had a modest ability to predict whether or not a CpG site was likely to be identified in an EWAS, and it did not add to the predictive ability of heritability, despite explaining a higher proportion of variance in effect estimates. This may be explained by two things. Firstly, having a lower variance in the independent or dependent variable increases the standard error of the beta coefficient in a linear regression. Secondly, heritability will in part determine variance of DNA methylation. 

### Choosing sites to measure {#choosing-sites-to-measure}

As discussed in __Section \@ref(ewas)__, the HM450 array was designed to capture DNA methylation in various regions of the genome. The probes of the array target roughly only 2% of CpG sites in the genome, yet target over 99% of protein coding genes and predominantly target the promoter regions of these genes [@Illumina2012]. The newer HMEPIC array captures much of what the HM450 array does, and further covers 58% of FANTOM5 enhancers [@Pidsley2016]. 

The trend for DMPs to be enriched for promoter regions (compared to regions of similar CG density) suggests there may have been some justification for the chosen sites. However, not all promoter regions were enriched with DMPs and bivalent promoters were depleted for DMPs. Enrichment of enhancers was also seen, but the magnitude of enrichment was smaller. When designing future arrays, these results suggest that continuing to target promoters and enhancers, whilst avoiding gene regions that are less likely to be actively transcribed may yield more associations in EWAS.

Despite the tissue specific nature of DNA methylation, the regions for which DMPs identified in The EWAS Catalog were found to be enriched were fairly consistent across tissues. However, enrichment of DMPs tended to be greater for blood-based genomic annotations, perhaps reflecting the fact the majority of EWAS in The EWAS Catalog were conducted using DNA methylation measured in whole blood. 

### Limitations {#limitations-04}

Individual participant data were not available and thus to calculate standardised betas, the variance of the trait had to be estimated from external measures of DNA methylation. If the GoDMC sample is not representative of the sample used for the study EWAS then these estimates may be substantially biased. Further, many studies do not report the effect estimates from their statistical analyses. If there is a marked difference in the studies that do not report effect sizes and those that do, then any associations between standardised effect estimates and DNA methylation site characteristics are likely to be biased. 

Like other observable phenotypes, DNA methylation varies under many contexts. Age, sex, tissue type, population, socioeconomic position and many other factors may influence the results of EWAS [@Birney2016; @Relton2010; @Relton2012]. The majority of EWAS conducted have used DNA methylation measured in whole blood from European adults making the results not necessarily apply broadly outside those bounds. The need for tissue-specific data has been discussed previously in __Section \@ref(problems-for-ewas)__. Differences in DNA methylation between ethnic groups has been shown previously [@ToinetCronje2020] and the predictive value of a smoking-related methylation score was shown to differ between Europeans and South Asians [@Elliott2014]. This suggests any biological insight and population health benefits that may be the result of EWAS is likely to to be maximised by diversifying populations. It is unclear from the work in this chapter whether the CpG characteristics and genomic annotations that show evidence that they influence EWAS results, will also influence EWAS results in the same way within a more ethnically diverse selection of samples.

## Conclusion {#conclusion-04}
This chapter demonstrates the potential for using large-scale EWAS databases to understand DNA methylation-trait associations. It was found that study design flaws can help explain some associations. However, it is noteworthy that the vast majority of studies have accounted for some potential biasing factors, for example `r comma(sum(batch_tab$batch_included & batch_tab$cell_comp_included) / nrow(batch_tab) * 100)`% of studies adjusted for batch effects and cell composition. Further, there was an invese association between DNA methylation variability and effect size, suggesting that studies that remove variable sites prior to analysis could be excluding important regions from the analysis. Finally, `r h_cpgs[length(h_cpgs)]` _`r names(h_cpgs)[length(h_cpgs)]`_ was identified as being associated with `r max(h_genes$n_traits)` traits that share known biological relationships. This highlights the potential to use The EWAS Catalog to identify molecular markers that might underlie the relationship between traits.
