# A comparison of the genes and genesets identified by EWAS and GWAS of eight complex traits {#ewas-gwas-comp-chapter}

```{r call_source-06}
read_chunk("chunks/06-ewas_gwas_comparison.R")
```

``` {r load-data-06, results="hide", message=FALSE}
```

``` {r trait-data-setup-06, results="hide", message=FALSE, warning=FALSE}
```

``` {r gwas-predicting-ewas-setup, results="hide", message=FALSE, warning=FALSE}
```

``` {r methods-sims-setup, results="hide", message=FALSE, warning=FALSE}
```

``` {r empirical-results-setup, results="hide", message=FALSE, warning=FALSE}
```

``` {r arch-sims-setup, results="hide", message=FALSE, warning=FALSE}
```

``` {r all-correlations-setup, results="hide", message=FALSE, warning=FALSE}
```

## Chapter summary {#chapter-summary}
Despite little evidence that DNA methylation correlates highly with complex trait variation (__Chapter \@ref(h2ewas-chapter)__), it is of interest to understand whether the DNA methylation sites identified may be valuable in understanding the underlying biology of complex traits. As discussed previously, it is well understood that observational associations between DNA methylation and complex traits are sucesptible to confounding and reverse causation. Genetic variants identified in GWAS will not be caused by phenotypic variation and their associations with complex traits are unlikely to be confounded. Yet, both GWAS and EWAS aim to identify regions of the genome that will give an indication of the molecular nature of complex traits. 

Using data from The EWAS Catalog (__Chapter \@ref(ewas-catalog)__) and the IEU OpenGWAS Project [@Elsworth2020; @Hemani2018], in this chapter I examine whether EWAS is likely uncovering similar facets of complex trait aeitiology to GWAS. 

## Introduction {#introduction-06}
Often in EWAS, the potential biological implications of differentially methylated positions or regions (DMPs or DMRs) will be investigated further through genomic annotations [@Sharp2017; @Reese2019; @Everson2019; @Chen2020]. As discussed in __Section \@ref(dna-methylation)__, previous studies have demonstrated a relationship between DNA methylation levels and proximal genes [@Jones2012; @Illingworth2009]. This observation has lead to it being common place to map sites identified in EWAS to nearby genes and these genes and their function are often probed to ascertain their relevance to the trait of interest [@Sharp2017; @Reese2019; @Everson2019; @Chen2020]. Further, genes can be grouped with others into "genesets" that have similar functionality or lie within the same pathway. Over-represented genesets may provide an insight into the molecular biology of a trait, for example, DNA methylation sites identified in an autoimmune disease EWAS might tag genes within immunological pathways more than expected by chance. Other assessments may be made to infer potential biological understanding, including enrichment of other epigenetic marks at the regions identified [@Breeze2016] and follow-up experimental studies [@Rakyan2011]. However, using open access databases to investigate tagged genes and genesets is a simple and potentially effective approach to further biological understanding. 

GWAS often use similar approaches to EWAS to help infer function from the signals identified [@Rakyan2011; @Gallagher2018]. However, the properties of genetic variants and DNA methylation differ making potential inferences from EWAS and GWAS diverge. Importantly, DNA methylation is responsive to environmental stimuli, thus making associations identified in EWAS potentially attributable to reverse causation and to confounding [@Birney2016; @Relton2010; @Relton2015]. It should be noted GWAS may be susceptible to confounding, but it is reasonable to assume it is less pervasive amongst GWAS associaitons [@DaveySmith2003; @DaveySmith2014]. 

Given what is known about the properties of genetic variants, a direct comparison between EWAS and GWAS results can provide insight into what biological information EWAS is capturing. If EWAS are highlighting a similar set of genes and genesets, it suggests changes in DNA methylation are identifying facets of trait aetiology. In the event that GWAS and EWAS are not highlighting similar genes and genesets, it is plausible that EWAS may still be identifying facets of trait aetiology. If DNA methylation mediates non-genetic effects and if sites are mapped to genes or genesets incorrectly then overlap will not be guarenteed even if DNA methylation identified is aetiologically relevant. When DNA methylation mediates the effect of genetic variants distal to their genomic position on complex traits, the genes identified by GWAS and EWAS will also differ, but the genesets would likely overlap. The lack of overlap could also reflect the fact associations in EWAS may be driven by confounding and reverse causation. Despite the caveats mentioned, one might still expect to detect overlap in genes and genesets identified by EWAS and GWAS of corresponding traits in the absence of confounding and reverse causation. The extent to which this expectation holds is discussed in more detail in the __Section \@ref(discussion-06)__.

In this chapter, the overlap between genes and genesets identified by GWAS and EWAS of `r num_to_text(nrow(trait_dat_tab))` complex traits is explored and the scenarios in which one would and would not expect to find overlap are modelled through simulations. 

## Results {#results-06}

### Study data {#study-data}
Traits were selected for which EWAS had been conducted with over 4500 samples and for which corresponding GWAS summary data were available. At the time of conducting these analyses, traits were identified using data from The EWAS Catalog (__Chapter \@ref(ewas-catalog)__) that contained EWAS published before 2019. The EWAS of glucose and insulin were also added at a later date. Traits and study data information is in __Table \@ref(tab:trait-data-tab-06)__.

```{r trait-data-tab-06, message=FALSE, results="asis"}
```

### Genomic position overlap {#genomic-position-overlap}
The genome was divided into `r max(div_dat_groups$group_total)` 500kb non-overlapping regions and mapped EWAS and GWAS signals for each trait to these regions (see __Section \@ref(methods-06)__ for more details). The number of regions that were identified by one study type and not the other was higher for each trait than the number of overlapping regions (__Figure \@ref(fig:overlap-barplot)__). Further, the magnitude of the greatest GWAS effect estimate in each region had little ability to predict whether or not a DNA methylation site was likely to be identified in the same region (AUC range = `r auc_range`, __Figure \@ref(fig:auc-plot)__). 

(ref:position-overlap) __Overlap between genomic positions identified by corresponding EWAS and GWAS__. The genome was divided into 500Kb regions. Those where no probes on the HM450 array measured DNA methylation were excluded from the analysis. Regions were counted as being identified by a GWAS if one or more SNPs associated with the trait and as being identified by an EWAS if one or more CpGs associated with the trait. Neither = no EWAS or GWAS sites identified in the region, GWAS = GWAS sites only were identified, EWAS = EWAS sites only were identified, Both = Both EWAS and GWAS sites were identified, AC = alcohol consumption per day, BMI = body mass index, CRP = c-reactive protein, CsNs = current smokers vs never smokers, EA = educational attainment, FsNs = former smokers vs never smokers. 

(ref:position-overlap-scap) Overlap between genomic positions identified by corresponding EWAS and GWAS

```{r overlap-barplot, message=FALSE, results="asis", fig.cap = '(ref:position-overlap)', fig.scap='(ref:position-overlap-scap)'}
```

(ref:auc-plot-cap) __Can genetic variant associations predict the presence of DNA methylation associations in the same region?.__ For each 500kb region in the genome, the largest SNP-trait effect size was extracted. ROC curves were produced to determine whether these could predict whether a differentially methylated position related to the same trait was present in the same 500kb region. The area under these curves (AUC), with their confidence intervals, are plotted for each trait. The red dashed line is at AUC = 0.5, which represents a prediction no better than chance. AC = alcohol consumption per day, BMI = body mass index, CRP = c-reactive protein, CsNs = current smokers vs never smokers, EA = educational attainment, FsNs = former smokers vs never smokers. Note: insulin is not present in this plot as there were GWAS and EWAS signal that overlapped across all the 500kb regions.

(ref:auc-plot-scap) Can genetic variant associations predict the presence of DNA methylation associations in the same region?

```{r auc-plot, message=FALSE, results="asis", fig.cap='(ref:auc-plot-cap)', fig.scap='(ref:auc-plot-scap)'}
```

### Assessing power to detect shared annotations between GWAS and EWAS {#assessing-power-sims}
Genomic function and trait biology are not divided into discrete 500kb genomic chunks, thus GWAS and EWAS could still be identifying similar facets of trait biology without identifying the same genomic regions. 

I sought to assess whether the genes and genesets identified overlapped more than expected by chance, thus genomic positions were mapped to genes and genes to genesets (details in __Section \@ref(methods-06)__). Overlap between genes identified was assessed using Fisher's exact test. Two methods for testing overlap between genesets were considered, one simply mapped genes to genesets and used Fisher's exact test in the same way as assessing overlap between identified genes. The other generated 'enrichment scores' for each geneset and assessed correlation between the geneset enrichment scores across studies. 

Assuming all genetic effects on DNA methylation are proximal. If all DMPs that associated with a trait were on the pathway from SNP to disease, then EWAS and GWAS would be identifying genes from the exact same geneset (i.e. genes that caused changes in the trait). The more DMPs that are identified because of confounding effects or reverse causation, the smaller the chance of overlap, assuming that causal and responsive genesets are distinct. As mentioned, the presence of trans-mQTLs would make the overlap of genes identified by GWAS and EWAS less likely despite both studies identifying aetiologically relevant factors. However, geneset overlap will be more likely to be maintained regardless of trans-mQTL presence. In a scenario where no DMPs are causing phenotypic changes, any overlap in genes and genesets found would be entirely attributable to chance (ignoring potential feedback loops whereby the trait causes changes in genesets that it is affected by). Simulations were run to assess which scenarios the enrichment and annotation methods had power to detect whether there was more overlap than expected by chance. Power was also assessed across different annotation methods. A schematic of how the simulations were set up can be found in __Figure \@ref(fig:method-simulations-schematic)__.

(ref:sim1-schematic-cap) __Flowchart demonstrating how the first set of simulations were set up__. The flowchart (under the title "Generating simulated data") shows how the simulated data were generated. The boxes under "Analysis" show the analyses performed with the simulated data. Underlined parameters were varied. The simulations were repeated 1000 times for each set of parameters. 

(ref:sim1-schematic-scap) Flowchart demonstrating how the first set of simulations were set up

```{r method-simulations-schematic, message=FALSE, results="asis", fig.cap = "(ref:sim1-schematic-cap)", fig.scap="(ref:sim1-schematic-scap)"}
```

Under each scenario, the ability to predict whether EWAS were identifying, in part, the same set of genes as GWAS ('causal genes') compared to a random set of genes ('associated genes') was tested. As expected, predictive capacity increased as study power and the proportion of causal DMPs increased (__Figure \@ref(fig:sim1-summ-plot)__, __Figure \@ref(fig:sim1-full-plot5)__). Predictive ability tended to increase as the number of identified genes increased, but this parameter was largely inconsequential when the proportion of causal DMPs was low (__Figure \@ref(fig:sim1-full-plot5)__). Performance was similar across annotation methods and between methods attempting to assess geneset overlap, with assessment of gene overlap performing better (__Figure \@ref(fig:sim1-summ-plot)__). Overall there was more power to detect overlap in genes than overlap in genesets. Between the geneset methods, there was more power to detect correlation between enrichment scores than direct overlap in genesets. Therefore, gene overlap and correlation between geneset enrichment scores were taken forward for the empirical analyses. Assessing correlations of GO term genesets had slightly more power (__Figure \@ref(fig:sim1-summ-plot)__) than other geneset databases, so these geneset annotations were taken forward.

(ref:sim1-sum-plot-cap) __Power to detect overlap between genes and genesets identified by corresponding EWAS and GWAS.__ Simulations were set up as illustrated in __Figure \@ref(fig:method-simulations-schematic)__. The area under receiver operator curves (AUC) was used to estimate the ability to distinguish between results generated when EWAS and GWAS were sampling, in part, from the same set of causal genes and results generated when EWAS were sampling random genes from the genome. The header of each set indicates the proportion of genes identified by the simulated EWAS that were set to be causal. or_g = assessing overlap of genes, or_p = assessing overlap of genesets, rho_p = assessing correlation between geneset enrichment scores. go = gene ontology, ppi = protein-protein interaction database from EpiGraphDB. This is a summary of the results, full results can be found in __Figure \@ref(fig:sim1-full-plot5)__.

(ref:sim1-sum-plot-scap) Power to detect overlap between genes and genesets identified by corresponding EWAS and GWAS

```{r sim1-summ-plot, message=FALSE, results="asis", fig.cap = '(ref:sim1-sum-plot-cap)', fig.scap='(ref:sim1-sum-plot-scap)'}
```

### Gene and geneset overlap between GWAS and EWAS {#gwas-ewas-overlap}

For the `r num_to_text(nrow(trait_dat_tab))` traits used, the number of genes identified by EWAS and GWAS that overlapped was low and for `r num_to_text(n_traits_no_g_overlap)` traits no genes identified by the studies overlapped (__Table \@ref(tab:empirical-gene-tab)__). The number of genesets that overlapped was higher, peaking at `r max_p_overlap` for GWAS and EWAS of `r mpo_trait` (__Table \@ref(tab:empirical-pathway-tab)__). 

The number of overlapping genes identified was no more than expected by chance for each trait (__Table \@ref(tab:empirical-gene-tab)__). There was also little evidence that correlation between enrichment scores of the GO terms was greater than expected by chance (__Table \@ref(tab:empirical-pathway-tab)__). As suggested by the previous simulations, this does not preclude the possibility that EWAS are identifying changes in DNA methylation that are upstream of phenotypic change. \linebreak

```{r empirical-gene-tab, message=FALSE, results="asis"}
```

```{r empirical-pathway-tab, message=FALSE, results="asis"}
```

\linebreak 

There were `r enriched_pathways_out$n_common` GO term genesets that were commonly enriched (FDR < 0.1) for both the EWAS and GWAS traits. Of these `r num_to_text(enriched_pathways_out$n_common_specific)` were specific (contained under 100 genes). There were `r enriched_pathways_out$n_uncommon_specific` specific genesets (geneset size < 100 genes) that did not overlap between studies of corresponding traits, for example, the genes identified by the GWAS of alcohol consumption were enriched for the “ethanol catabolism” pathway (geneset size = 12 genes), however none of the genes identified by the EWAS were present in this pathway. 

### Understanding architecture from geneset overlap {#architecture-sims}

The total number of genes and genesets that can be identified by GWAS and EWAS are unknown and the proportion of causal and non-causal genes EWAS will identify is also unknown. Thus it is uncertain how much overlap there will be between genes and genesets of corresponding GWAS and EWAS as sample sizes increase and more genes and genesets are discovered. Using the empirical data from __Table \@ref(tab:empirical-pathway-tab)__ to inform simulations, I sought to identify the proportion of overlap between causal and associated genes (genes that may or may not be causal) and how this is related to the number of genes yet to be discovered.

For the simulations, three sets of genes were linked to each trait: genes identified by the GWAS (known GWAS genes), genes identified by the EWAS (known EWAS genes) and a random set of genes sampled from the total set of Ensembl gene IDs (excluding the genes identified by the EWAS and GWAS). Those genes identified by the GWAS and a number of the randomly selected set of genes was assigned as "causal" and the genes identified by the EWAS and the rest of the randomly selected set of genes was assigned as "associated". The overlap between the causal and associated genes as well as the number of total genes (causal and associated) was pre-determined. From these pools of causal and associated genes, $X$ genes were sampled randomly to represent GWAS genes and $Y$ genes were sampled randomly to represent EWAS genes. The number of genes, $X$ and $Y$, sampled was equal to the number identified in the empirical GWAS and EWAS, respectively.

Having generated the GWAS and EWAS genes, enrichment of GO terms was performed and the correlation between enrichment scores across all the terms was estimated. Under a single simulation scenario, the number of total genes and the proportion of causal and associated gene overlap was pre-determined. The known GWAS and EWAS genes was determined by those discovered from the empirical results, thus the random set of genes was varied to alter the total number of genes and proportion of overlap. The proportion of overlap was 0, 0.01, 0.1, 0.5 or 1 and the total number of genes was split evenly into causal and associated genes. The number of associated genes (and causal genes) equalled the total number of EWAS and GWAS genes discovered multiplied by 1, 2, 3, 5, 10 or 20. For each scenario, the analyses were repeated 1000 times for each trait. A schematic of the methods for these simulations can be found in __Figure \@ref(fig:arch-simulations-schematic)__ and it is described in full in the __Section \@ref(methods-06)__. 

(ref:arch-simulations-schematic-cap) __Flowchart demonstrating how the second set of simulations were set up for each trait.__ Phenotypic variation may be caused by changes in gene/protein polymers (causal genes) and can be associated with changes in gene/protein polymers via other routes such as confounding or reverse causation (associated genes). In these simulations the causal genes were a mix of genes identified by GWAS, known GWAS genes (KGG), and a random set of genes, simulated GWAS genes (SGG). The associated genes were a mix of genes identified by EWAS, known EWAS genes (KEG), and a random set of genes, simulated EWAS genes (SEG). The level of overlap in the causal and associated genes was modified by changing the overlap in the SGG and SEG. The number of causal and associated genes was kept the same for each simulation, but this number varied between simulations. The minimum number of causal genes and the minimum number of associated genes was equal to the sum of KGG and KEG. The "simulated GWAS" step in the simulation simply equates to randomly sampling from the causal genes. The number of genes sampled was equal to the number of KGG. The "simulated EWAS" step was identical except the number of KEG from the associated genes. Geneset enrichment was performed as described in __Section \@ref(methods-06)__. The simulations were repeated 1000 times for each set of parameters. 

(ref:arch-simulations-schematic-scap) Flowchart demonstrating how the second set of simulations were set up for each trait

```{r arch-simulations-schematic, message=FALSE, results="asis", fig.cap = '(ref:arch-simulations-schematic-cap)', fig.scap='(ref:arch-simulations-schematic-scap)'}
```

From the simulations it was determined that, for seven of eight traits, it is likely that if the number of genes discovered currently is near to the total genes to discover for these traits, the proportion of overlap between causal and associated genes is low. However, for former vs. never smoking, there was greatest evidence for a scenario where the causal and consequential gene overlap was high. The results from the analysis of former vs. never smoking and c-reactive protein (representing simulations for the other traits) are shown in __Figure \@ref(fig:arch-simulations-crp-fvns)__. __Figure \@ref(fig:arch-simulations-supp-res)__ shows the same results for the other traits. Each simulation was repeated 1000 times. Evidence for a difference in the empirically determined correlation of geneset enrichment scores and the mean correlation of geneset enrichment scores across simulations was assessed using a z-test for difference. There was some evidence against `r nrow(asim_ztest_out)` simulation scenarios (FDR < 0.05). Across the traits, the scenarios that were least likely tended to be when the number of genes yet to discover was low, and the overlap between causal and associated genes was high, except for former vs. never smoking, highlighting architecture differences between traits. 

(ref:arch-simulations-crp-fvns-cap) __Simulations to understand the likely number of genes still to identify in EWAS and GWAS of c-reactive protein and former vs. never smoking under different trait architectures__. Simulations were set up as illustrated in __Figure \@ref(fig:arch-simulations-schematic)__. Correlation of geneset enrichment scores from empirical data (__Table \@ref(tab:empirical-pathway-tab)__), is shown as a red dashed line. Box plots show the range of enrichment score correlations from 1000 simulations using the parameters indicated. The number of causal and associated genes, as well as the overlap between these genes were varied. N~KEG~ = number of known EWAS genes, N~KEG~ = number of known GWAS genes, N~KTG~ = number of known total genes (N~KEG~ + N~KGG~). By way of an example, when N~KTG~ = 491 and the ratio of causal and associated genes relative to N~KTG~ is 1:1, the number of causal genes in the simulations will be 491 and the number of associated genes in the simulations will be 491. Scenarios which lie close to the empirical result (red dashed line) are more likely to reflect the true underlying number of genes related to a trait and the true overlap between the causal and associated genes. Where there is some evidence that the empirical results are different to the simulations (z-test FDR < 0.05) the box outline is grey.

(ref:arch-simulations-crp-fvns-scap) Simulations to understand the likely number of genes still to identify in EWAS and GWAS of c-reactive protein and smoking (former vs. never smokers) under different trait architectures

```{r arch-simulations-crp-fvns, message=FALSE, results="asis", fig.cap = '(ref:arch-simulations-crp-fvns-cap)', fig.scap='(ref:arch-simulations-crp-fvns-scap)', out.width='100%'}
```

### Overlap of non-corresponding EWAS and GWAS {#matching-gwas-to-ewas}
It may be the case that EWAS of one trait are actually more closely related to GWAS of another trait. For example, if DNA methylation changes related to BMI were mediating the effect of BMI on changes in metabolites, then one could expect to see greater correspondence in identified genes and genesets with GWAS of those metabolites.

To test if this was the case for any of the EWAS in this study, `r nrow(ieugwas_studies)` GWAS with SNPs associated at 5x10^-8^ with a sample size above 5000 from the IEU OpenGWAS Project [@Elsworth2020] were extracted. For each GWAS and the `r num_to_text(nrow(trait_dat_tab))` EWAS, enrichment scores were calculated using geneset enrichment analysis and correlation between the enrichment scores was calculated, as in previous analyses. 

Across all pairwise comparisons, correlations between enrichment scores ranged from `r comma(cor_range[1])` to `r comma(cor_range[2])` and had a mean of `r comma(cor_mean)`. The mean correlation between GWAS traits and EWAS traits was `r comma(ewas_gwas_mean)`, which was higher than the correlations between just GWAS traits (`r comma(gwas_gwas_mean)`). Amongst just GWAS results, there was evidence that `r nrow(gwas_cor_tab)` pairwise enrichment score correlations were greater than the the mean (FDR < 0.05). However, there was little evidence that any pairwise correlations between enrichment scores derived from EWAS and GWAS were greater than the mean correlation (FDR > 0.05). 

This suggests that the signal from EWAS is not capturing aspects of any specific factor that impacts the aetiology of any of the `r num_to_text(nrow(trait_dat_tab))` traits of interest. 

## Discussion {#discussion-06}
Several EWAS papers have compared their findings with those of the corresponding trait GWAS [@Kurushima2019; @Everson2019; @Agha2019; @Odintsova2019; @Geurtsen2019], but it's unknown if any overlap that might occur should be attributed to shared underlying genetic and epigenetic architectures or if it occurs by chance. In this study, the genes and genesets identified by `r num_to_text(nrow(trait_dat_tab))` large EWAS (N>4500) were not identified in their corresponding GWAS any more than expected by chance. Simulations suggested these EWAS could still be identifying aspects of trait aetiology, but it is likely most DMPs identified are due to confounding or reverse causation. Further simulations suggested that the overlap between genes that impact phenotypic variation and those that might be identified through confounded analyses or reverse causation is likely to be low. However, if the number of genes still to identify in EWAS and GWAS is high, it is possible that the overlap between "causal" and "associated" genes could be high also. 

### Overlap expected {#overlap-expected}
GWAS identifies the effects of genetic variation on complex traits. These effects are less likely to be confounded than associations estimated between observational phenotypes [@DaveySmith2003; @DaveySmith2014]. Thus, one would expect overlap between genes and gemesets identified by EWAS and GWAS of the same trait if the DMPs identified are also of aetiological relevance. Assuming mapping of DMPs and SNPs to genes is correct, the genes identifed by EWAS may cause variation in complex traits without overlapping with genes identified in GWAS. Under the scenario where the effect of a SNP on a complex trait is mediated by a distal DNA methylation site (or the gene that site is tagging), GWAS and EWAS may identify genes that do not overlap, but that are along the same causal pathway from genome to complex trait. One plausible mechanism for which trans-mQTLs may act is via transcription factors. A trans-mQTL may influence the transcription of a nearby transcription factor then the transcription factor could cause a change in DNA methylation at distal sites. One study has provided evidence this may occur frequently [@Bonder2017]. In this scenario the genes proximal to the identified SNP and DMP would lie on the same causal pathway and so would likely be part of the same genesets. This is likely not the only plausible mechanism of trans-mQTL function though. DNA methylation may also mediate non-genetic effects, allowing for causal DMPs to be identified at genes not near pertinent genetic variation. However, there is strong evidence that the majority of DNA methylation sites have a heritable component. As such any effect of DNA methylation on a trait could be influenced by genetic variation. If DNA methylation is influenced by proximal genetic variants then the discovery of the same gene(s) will be a function of GWAS and EWAS power. If only distal genetic variants influence the DNA methylation site, then the overlap of genesets is a function of regulatory mechanisms and power. These two issues, and likely others, may introduce noise into the results. However, there is overwhelming evidence that confounding and reverse causation are pervasive across observational epidemiology [@Lawlor2004; @Sattar2017; @DaveySmith1992; @Fewell2007] as well as within EWAS [@Birney2016; @Heijmans2012]. This suggests that the evidence that EWAS and GWAS are not identifying any more overlapping genes and genesets than expected by chance, is likely due to identified DMPs being mostly the result of reverse causation or confounding.

As the simulations showed, even if an EWAS identifies DMPs that cause a change in the trait, if the majority of DMPs identified are due to confounding or reverse causation then the overlap will be indistinguishable to the overlap expected by chance. Thus, our empirical results do not preclude the possibility that some DMPs identified by the EWAS are pertinent to trait aetiology.

For body mass index, work has already suggested the traits cause changes in DNA methylation rather than _vice versa_ [@Philibert2016; @Wahl2017], supporting our findings here. Further, one study suggested DNA methylation changes capture different components of body mass index variance than genetic variation [@Shah2015] and another estimated the percentage of trait variance captured by DNA methylation was 75.7% when accounting for genotype [@TrejoBanos2020]. 

Simulations involving empirical data from former vs. never smoking EWAS and GWAS suggested that overlap between causal and associated genes was high. This is surprising for two reasons. Firstly, it differs from the current vs. never smoking results, suggesting distinct genetic or epigenetic architectures of those traits. Secondly, there is evidence that for DNA methylation changes identified in relation to smoking, smoking is likely causing DNA methylation variation and not _vice versa_. Although the statistical tests suggested that it was unlikely that the proportion of causal and associated genes is 0.1 or lower, it should be noted that the absolute difference between simulated results and empirical results were not great. 

If EWAS is discovering some DMPs that influence genes that cause changes in the trait of interest, study power is the limiting factor for detecting overlap. For traits with a weak polygenic architecture (few genes explain most of the heritability), such as gene expression [@Wheeler2016; @Mogil2018], discovering almost total overlap would be inevitable even with modest sample sizes.

### Little overlap with any GWAS {#little-overlap-with-any-gwas}
Little correlation was found between geneset enrichment scores for EWAS and GWAS of different traits. In a scenario where DNA methylation was capturing a specific facet of a trait one might expect correlation between EWAS of the original trait and GWAS of that facet. For example, if changes in DNA methylation associated with smoking were mostly responsible the effect of smoking on lung cancer then one would expect to observe an overlap between the genes and pathways identified by an EWAS of smoking and GWAS of lung cancer. This specific example has been examined before, with studies suggesting either methylation at two sites mediate over 30% of the effect of smoking on lung cancer [@Fasanelli2015] or that there is little evidence for a causal effect of DNA methylation on lung cancer [@Battram2019]. The results of either study suggest most sites will not mediate the effect of smoking on lung cancer and thus there would be little overlap between genes and pathways of an EWAS of smoking and a GWAS of lung cancer. This is corroborated by the results of our study: there was little evidence that correlation of pathway enrichment scores between the two, `r comma(lc_sm_rho)`, was greater than the mean correlation enrichment score across all GWAS-GWAS and GWAS-EWAS correlations, `r comma(cor_mean)` (FDR > 0.05).

It is important to note that overlap between EWAS and GWAS genesets may be missed even if this mediation model is true for various traits. As shown in the simulations, detecting this overlap depends on individual study power as well as the underlying genetic and epigenetic architecture of the trait. There are further things that may limit detection of geneset overlap that is discussed later on.

### Information gained from EWAS {#information-gained}
The fact that genes and pathways identified from GWAS and EWAS of the same traits are seemingly very separate suggests we are gaining new information from EWAS, even if interpreting the new information may be difficult. Key to interpreting the EWAS results would be to try and disentangle whether the EWAS results are likely due to confounding (as is explored in __Chapter \@ref(dnam-lung-cancer-mr)__). Interpreting EWAS can also be difficult due to cell type heterogeneity and the complexity of mechanisms which mediate DNA methylation changes [@Birney2016; @Jones2012; @Houseman2012; @Jaffe2014; @Hemani2017]. Due to these difficulties it should not be concluded that EWAS definitely help increase our biological understanding of complex traits. Rather, DNA methylation is capturing different biological information. Regardless of biological insight gained, translational impact may still be gleaned from DNA methylation studies; DNA methylation may aid diagnoses by acting as a reliable biomarker or could help predict various health outcomes [@Relton2010]. 

There are also benefits to understanding the biological consequences of a trait, something that EWAS might help identify and GWAS will not (at least not directly). This does depend on further research to understand how changes in DNA methylation downstream to complex trait variation is relevant to human health. Further, establishing where exactly DNA methylation may lie on the causal pathway may be difficult and work is ongoing to discover this for various traits [@Neumeyer2020]. Use of causal inference methods such as Mendelian randomization [@DaveySmith2014; @DaveySmith2003; @Relton2012] can be applied (and are in __Chapter \@ref(dnam-lung-cancer-mr)__), but this still comes with various caveats, as discussed in __Section \@ref(establishing-causality)__ [@Relton2012; @Neumeyer2020]. Some studies also try and confirm effects experimentally [@Rakyan2011] and use previous biological knowledge of the trait to try and understand EWAS results. 

However, some of the biological knowledge of complex traits relating to genes and pathways comes from GWAS of those traits. This study suggests that EWAS is unlikely to identify many genes proximal to genetic variation pertinent to the trait of interest and further the genesets are unlikely to overlap with those identified in GWAS. Therefore, comparison of EWAS results to those of a corresponding GWAS is unlikely to explain a large proportion of DNA methylation-trait associations observed in EWAS. This may make inference from EWAS difficult, yet it seems likely the interpretability of DNA methylation studies will continue to improve over the coming years, as understanding the underlying epigenetic architecture of complex traits could still provide translational benefits [@Relton2010]. 

### Limitations {#limitations-06}
As discussed, detecting gene or pathway overlap depends on the genetic and DNA methylation architecture of the trait. Here only `r num_to_text(nrow(trait_dat_tab))` traits, two of which are smoking behaviour traits, have been studied. This means the results cannot be generalised to all or even the majority of complex traits. These analyses could be repeated by setting a less restrictive sample size limit, but it was felt that would make the results less reliable and impossible in many circumstances where too few DNA methylation sites had been discovered by EWAS. As sample sizes increase and technologies measuring more DNA methylation sites become more common, it would be interesting to repeat the analysis.

Often in GWAS and EWAS, prioritisation of SNPs and DMPs identified occurs before functional mapping. Prioritsation for both studies may be informed by prior knowledge of the trait, prior understanding of molecular biology, predicted consequences of observed variation (for example using Ensembl's Variant Effect Predictor [@McLaren2016]), replication of findings or a number of other methods. In this study, I did not perform any prioritisation (besides the conventional P value threshold cutoffs) and thus may have increased the amount of "noise" in the signal taken forward for functional annotation. Unfortunately, this extra prioritisation of sites is not tractable when comparing many different association studies and may reduce power to detect any overlap between genes and pathways. However, added noise is unlikely to prevent detection of true overlap if that true overlap is substantial, as shown by the simulations (__Figure \@ref(fig:sim1-summ-plot)__).

The nearest gene, by chromosomal position, to a DMP or SNP is not necessarily the gene of interest. SNPs may have effects on genes distal to their position [@Aguet2017]. Also, the correlation between genetic variants inflates associations of variants proximal to the true causal variant, which may map to unrelated genes. Further, the correlation structure in DNA methylation data may induce associations between complex traits at a site far from where variation in DNA methylation causes complex trait changes [@Aguet2017]. Therefore, the mapping of DMPs and SNPs to genes in this study could likely be improved. The "correct" method for this mapping has not yet been established though and even though some tools are available (such as eQTL studies), there are caveats to them too [@Huang2018; @Kanduri2019]. 

Our understanding of molecular pathways is not complete and thus attributing genes to certain pathways or functionalities may be erroneous. However, the results remained consistent across four different methods that annotate genes to pathway, suggesting differences in mapping genes to genesets should not impact our conclusions.

Biological information gained from EWAS and GWAS may be defined in various ways and depending on the interpretation of this, one could alter methods used to extract biological information. However, first exploring the genomic regions identified and then mapping these to potentially relevant biological genesets is common amongst GWAS and EWAS [@Sharp2017; @Reese2019; @Everson2019; @Chen2020; @Yengo2018; @Visscher2017; @Ellinghaus2016] and gave a simple way to compare the information from the two study types.

## Conclusion {#conclusion-06}
Overall this chapter provides evidence that, for `r num_to_text(nrow(trait_dat_tab))` complex traits, there is little overlap between genes and genesets identified by EWAS and GWAS of the same trait. Given the differences in properties between DNA methylation and genetic variants the results presented in this study may apply to other traits, but this is still to be confirmed. Where lack of overlap between genes and genesets is found, it suggests EWAS may be providing new biological information, however, the interpretability of EWAS is still in question and with current methods it is hard to determine if EWAS results are attributable to confounding or reverse causation. Regardless, as causal inference within epigenetic epidemiology improves, it seems likely we will be able to interpret this apparent gain in biological information. 

## Methods {#methods-06}

### Samples {#samples-06}
EWAS summary data and GWAS summary data were extracted from the EWAS Catalog (__Chapter \@ref(ewas-catalog)__) and the IEU OpenGWAS Project [@Hemani2018; @Elsworth2020] respectively. The data was extracted in July 2019, when the EWAS Catalog contained data published before 2019. For traits that had multiple EWAS with a sample size of greater than 4500, the EWAS with the largest sample size was used, the same was applied to the GWAS. The sample size, first authors and PubMed IDs can be found in __Table \@ref(tab:trait-data-tab-06)__.

### Overlapping genomic regions {#overlapping-genomic-regions}
Each chromosome was divided into 500Kb blocks, each block that did not contain a DNA methylation site measured by the HM450 was removed. For each trait, the genome blocks that had one or more EWAS sites and one or more GWAS sites that reached the set p-value threshold were tallied. The p-value threshold was set at a lenient P < 1x10^-5^ or if it was lower, the maximum reported p-value in the EWAS of that trait.

### Mapping sites to genes and genesets {#mapping-sites-to-genes-and-genesets}
The R package biomaRt [@Durinck2009] was used to extract Ensembl gene ids along with chromosome positions of all genes. The package was also used to extract gene ontology (GO) terms [@Ashburner2000; @Carbon2019] and map these to the Ensembl gene ids. The R package limma [@Ritchie2015] was used to extract KEGG terms [@Kanehisa2000; @Kanehisa2019; @Kanehisa2019] and these were mapped to Ensembl gene ids. 

Protein-protein interaction data, which includes data from StringDB [@Szklarczyk2019] and IntAct [@Orchard2014], and terms from the Reactome database [@Jassal2020] were extracted from [EpiGraphDB](http://www.epigraphdb.org/) [@Liu2020]. 

CpG sites associated with traits at P < 1x10^-7^ and SNPs associated with traits at P < 5x10^-8^ were taken forward to be mapped to genes. The correlation structure present in genetic and DNA methylation data makes it difficult to ascertain the precise site driving any signal observed. Thus, no filtering based on correlation between variants or CpG sites was performed.

For each CpG site identified by EWAS and used in the analyses, it was mapped to the nearest gene (Ensembl gene ID) by chromosome position. If a CpG site lay within the bounds of multiple genes then the site was mapped to all of those genes. Therefore, one CpG site could map to multiple genes and one gene could map to multiple CpG sites. The same gene mapping approach was used for variants identified in GWAS. The positions of CpG sites were extracted using the R package meffil [@Min2018].

### Methods for assessing overlap {#methods-for-assessing-overlap}

To test the overlap between genes identified we generated ORs as follows

\begin{equation}
    OR_{gene-overlap} = \frac{odds_{EG}} {odds_{EnG}}
    (\#eq:overlap-of-genes)
\end{equation}

where $odds_{EG}$ is the odds of a gene being identified in EWAS and GWAS and $odds_{EnG}$ is the odds of a gene being identified in EWAS, but not in GWAS.

Genes may map to genesets by chance. Often in EWAS and GWAS, enrichment for any genesets are tested by assessing whether the genes identified are more common in any geneset than expected by chance. Power was compared between mapping genes to genesets and directly assessing overlap like in Equation \@ref(eq:overlap-of-genes) and correlation between enrichment scores for each geneset. Enrichment scores are also odds ratios generated in a similar way to those in Equation \@ref(eq:overlap-of-genes): 

\begin{equation}
    OR_{geneset-enrichment} = \frac{odds_{GS}} {odds_{nGS}}
    (\#eq:geneset-enrichment)
\end{equation}

where $odds_{GS}$ is the odds of a gene being annotated to the geneset and $odds_{nGS}$ is the odds of a gene not being annotated to the geneset.
 
For many genesets, genes annotated to that geneset would not be identified in an EWAS or GWAS, causing the enrichment score for many genesets to be zero. Further, some genesets would have very large enrichment scores. This made the relationship between enrichment scores generated for the EWAS results and the enrichment scores generated for the GWAS non-normal. Thus, to examine the relationship between the two, Spearman's rank correlation coefficients of the logarithm of enrichment scores was used.

### Testing power to detect overlap {#testing-power-to-detect-overlap}

Simulations were setup as seen in __Figure \@ref(fig:method-simulations-schematic)__. The simulations iterated over each set of parameters 1000 times. For each iteration, two sets of genes, GWAS genes and EWAS genes, were sampled from the total set of genes. Each iteration assessed gene overlap and geneset overlap between these gene sets using Equation \@ref(eq:overlap-of-genes). Equation \@ref(eq:geneset-enrichment) was used to generate enrichment scores for each gene set and then correlation between the enrichment scores was assessed. 

GWAS genes were only sampled from a set of "causal" genes and a proportion of EWAS genes were sampled from the set of causal genes and the rest from the set of "associated" genes. Receiver operator characteristic (ROC) curves were generated to assess whether it was possible to predict the gene overlap, geneset overlap and enrichment score correlations for scenarios where the proportion of causal EWAS genes was greater than zero from the scenario where the proportion of causal EWAS genes was zero. The area under these ROC curves were then calculated in each case.

These simulations were repeated for each geneset and protein-protein interaction database. The protein-protein interaction and Reactome databases all map only to protein coding genes, whereas the GO and KEGG databases map to all Ensembl gene IDs. To compare predictive ability across annotation methods, Ensembl gene IDs that were not protein coding genes were excluded. A comparison between the performance of models when mapping to all Ensembl gene IDs or to protein coding genes was made for GO and KEGG databases. (__\@ref(fig:sim1-go-kegg-gene-comp5__).

From these simulations, the best method to assess geneset overlap, and the best geneset annotation method to assess that overlap and the scenarios (i.e. study power required, proportion of DMPs that need to be causal) in which it could be expected to be able to detect overlap could be deduced.

### Empirical analyses {#empirical-analyses}
The DNA methylation sites identified in the EWAS at P < 1x10^-7^ and the SNPs identified in the GWAS at P < 5x10^-8^ were mapped to genes and genesets. Overlap between genes was calculated as before (Equation \@ref(eq:overlap-of-genes)), enrichment scores were generated and correlated as described above. 

Expected overlap was generated to compare to the observed results. For this, random positions were chosen in the genome equal to the number of DMPs identified in the EWAS. These genes were then used to assess gene and geneset overlap as for the observed results. This was repeated 1000 times to generate a null distribution and a z-test was used to assess whether there was a difference between the observed results and the mean of the null distribution. 

There is a correlation structure within DNA methylation data [@Zhang2015], and it was hypothesised this might contribute to the observed results. By randomly sampling positions from the genome, a new correlation structure between DMPs would be generated. I tested whether sampling the genome in a non-random way, aimed at keeping some correlation structure, altered the results. To generate new data whilst attempting to keep a similar correlation structure, a fixed number of base pairs were added to each of the DMPs identified in the empirical analysis such that

\begin{equation}
    BP_{new} = BP_{dmp} + max(G) \times I
    (\#eq:generating-new-positions)
\end{equation}

where $BP_{new}$ = base pair of new site, $BP_{dmp}$ = base pair of DMP identified in the EWAS, $G$ = gene size, and $I$ = iteration. 

If $BP_{new}$ extended beyond the end of a chromosome, the position moved onto the next chromosome, with positions moving past the end of chromosome 22, being moved to chromosome 1.

Overall, the overlap between genes and genesets identified by EWAS and GWAS did not change across null distribution sampling methods.

### Understanding architecture from geneset overlap  {#architecture-sims-method}

Simulations were setup as illustrated in __Figure \@ref(fig:arch-simulations-schematic)__. Here I describe simulations for a single trait. These were repeated for all traits. Firstly, SNPs identified in GWAS and DMPs identified in EWAS were mapped to genes as described in __Section \@ref(gwas-ewas-overlap)__. Genes were then randomly sampled from Ensembl gene IDs and were assigned as either "causal", meaning changes in that gene effect variation in the phenotype across individuals, or "associated", meaning changes in the gene are associated with the phenotype across individuals, but the nature of association is not known. The empirically identified (or "known") GWAS genes ($KGG$) were added to the list of causal genes and the empirically identified (or "known") EWAS genes ($KEG$) were added to the list of associated genes. These combined set of causal and associated genes can be thought of as all the genes related to the trait of interest. A number of genes, equal to the number of $KGG$ ($N_{KGG}$), was sampled from the causal set of genes and assigned to be the "GWAS genes" in the simulations. A number of genes, equal to the number of $KEG$ ($N_{KEG}$), was sampled from the associated set of genes and assigned to be the "EWAS genes" in the simulations. Then geneset enrichment analyses for both the EWAS and GWAS genes were performed (equation \@ref(eq:geneset-enrichment)) and correlation between the enrichment scores was assessed as previously. In these simulations, the number of total genes was varied and the number of causal and associated genes was always set to be half of the total number of genes related to a trait. The total number of genes was proportional to the total number of known genes ($N_{KTG} = N_{KGG} + N_{KEG}$). The smallest number of total genes for any simulation was double the number of $N_{KTG}$ and the greatest number of total genes was 40 times $N_{KTG}$. The other variable set to vary between simulations was the proportion of overlap between causal and associated genes. This varied between zero and one, where zero represented the scenario where only the overlap in $KGG$ and $KEG$ would be present in the overlap between causal and associated genes and one represented the scenario where all causal and associated genes were the same. For each simulation scenario, the simulations were repeated 1000 times and box plots show the range of output from those 1000 repeats.

### Assessing the correlation between geneset enrichment results {#assessing-the-correlation-between-geneset-enrichment-results}
GWAS were extracted from the IEU OpenGWAS Project [@Elsworth2020] with the following criteria:

* Sample size > 5000
* European population
* For binary traits, cases and controls had to be greater than 500
* Full genome-wide results, i.e. not just associations between a molecular trait and variants in cis.

For each GWAS, all SNPs that associated with the trait at P<5x10^-8^ were extracted. CpGs associated with the `r num_to_text(nrow(trait_dat_tab))` EWAS at P<1x10^-7^ were then extracted and mapped to genes. For each study, enrichment scores were generated for GO terms as before (Equation \@ref(eq:geneset-enrichment)) and correlation between them assessed. 

When assessing whether gene overlap or geneset enrichment score correlations were greater than the mean, a z-test was performed. As multiple tests were performed we applied the Benjamini-Hochberg method [@Benjamini1995] to limit the false discovery rate. 

