# Discussion {#discussion-thesis}

## Overview {#overview-08}

Phenotypic variation cannot occur without molecular change. Identifying the changes that cause phenotypic variation adds to the aetiological understanding of traits and thus has the potential to uncover novel therapeutic targets. Further, studying the association between molecular marks and complex traits may yield new, valid predictors, that could augment current prediction capacity within clinical practice [@Relton2010]. This is true regardless of whether the molecular marks identified are causal.

Given the strong links made by experimental researchers between DNA methylation and the regulatory processes in cells [@Suzuki2008; @Siegfried1999; @Bird2002; @Jones2012; @Ando2019; @Deaton2011; @Wolf1984; @Hellman2007], there has been precedence for studying the covariation between DNA methylation and complex traits with the hope of discovering new regulatory features underlying trait biology. 

Over the past 15 years, hundreds of EWAS have been conducted, but the number of associations identified have varied dramatically by trait. EWAS have identified numerous associations between DNA methylation and complex traits such as smoking [@Joehanes2016; @Shenker2013; @Zeilinger2013] and body mass index [@Mendelson2017; @Wahl2017; @Shah2015; @Demerath2015]. However, there are also many examples where EWAS have identified few sites that reliably associate with a complex trait; in The EWAS Catalog (__Chapter \@ref(ewas-catalog)__), roughly 25% of the EWAS report fewer than 10 associations. Given the phenotypic nature of DNA methylation, one might expect to see covariation of complex traits and DNA methylation to be identified in abundance across the majority of EWAS. This has not been the case thus far. Understanding the aspects of EWAS that can explain associations across complex traits is imperative to progressing study designs and was the focus of this thesis.

In this body of work hundreds of published EWAS along with hundreds more EWAS I conducted were catalogued in an open access database. Using these results, I showed that unaccounted for technical effects may explain over 10% of reported associations. When accounting for these technical effects, the total predictive capacity of DNA methylation measured using the HM450 array in whole blood was low across 400 complex traits (median was near zero). This did vary substantially between traits, yet having smoked cigarettes was the only trait for which there was evidence that the total predicitive capacity was greater than zero (FDR < 0.1). This suggests that the lack of association between DNA methylation (as measured in blood by the HM450 array) and some complex traits is unlikely due to power, and we should not expect to see similar levels of wide-spread pertubations of DNA methylation in relation to traits like with smoking, when similar sample sizes for analysis are obtained. This apparent lack of covariation between DNA methylation and complex traits may be surprising given the phenotypic nature of DNA methylation, but as this study (as with most EWAS) only measured methylation at less than 2% of the total number of CpG sites across the genome and in only one tissue, it is plausible that much of the covariation between DNA methylation and complex traits was missed. Despite evidence of little predictive capacity, there are still EWAS that have identified numerous DMPs. However, it was found that EWAS for eight traits identified no more genes and genesets in common with corresponding GWAS than expected by chance, suggesting that the majority of DMPs identified were either acting through pathways independent of genetics or were not of aetiological relevance for the traits examined. Confounding and reverse causation may explain the marked differences in EWAS and GWAS results of corresponding traits. In the specific scenario of an EWAS of lung cancer, the extent to which DNA methylation changes may not be on the causal pathway to traits was examined. Observational EWAS of lung cancer identified associations with large effect sizes between DNA methylation, measured long before disease onset, and lung cancer risk. However, in a well powered Mendelian randomization (MR) analysis, none of these associations could be replicated. By way of example, the strongest observational association found was at cg05575921, OR = 0.47, and under the MR framework OR = 0.943. This suggests that, even with large effect sizes and an observational study desinged to limit the impact of reverse causation and confouding, identified EWAS associations may not be causal.

This work also revealed some promising aspects of the current EWAS study design. Firstly, the regions targeted by probes on the HM450 array were designed to be enriched for promoters of protein-coding genes and work presented in __Chapter \@ref(properties-of-ewas)__ suggests DMPs are also enriched for these regions. Therefore, this decision may have increased likelihood in identifying sites associated with complex traits. Secondly, the evidence that EWAS are identifying separate genes and genesets to GWAS does not preclude the possibility that these EWAS associations are useful. In fact, if overlap was very high then it would suggest a large amount of redundancy in the studies, but instead it is likely the study types are capturing very different information. 

Overall though, these findings suggest that substantial changes may be required in the design of EWAS to improve predictive capacity and enhance aetiological insight. 

## Extensions to this work {#extensions-to-work}

Indeed the issues pertaining to some design flaws and potential improvements that can aid interpretation of EWAS has already been reviewed [@Birney2016; @Rakyan2011; @Heijmans2012; @Lappalainen2017]. Lappalainen & Greally also suggested moving to conducting "second generation EWAS", which focused on improving interpretability of EWAS, especially when conducting them as a means to understand trait biology [@Lappalainen2017]. They argue that there are four key aspects which should be the focus for future EWAS. Firstly, the hypothesised method for which cellular changes will result in phenotypic variation should be clear. Secondly, participants and cell samples collected should be obtained to answer necessary questions regarding causality. Thirdly, cell types present in samples should be measured as precisely as possible. Finally, as many complementary genome-wide assays as possible should be conducted in the same samples and these assays should survey the maximum amount of loci possible. Reflecting on the work presented in this thesis, and following on from these points, here I discuss ideas that could be applied to improve both yield and interpretation of EWAS.

### Beyond the HM450 array in blood {#beyond-the-hm450}

The work in this thesis focuses on DNA methylation measures taken for the majority of published EWAS, namely the use of the HM450 array to measure DNA methylation in blood samples. The HMEPIC array is now regularly used, but as will be explained, many of the limitations of EWAS identified in this thesis apply to studies using the contemporary array technology. However, given the correlation structure across all DNA methylation sites in the genome is not known, this work can not be generalised to DNA methylation studies as a whole. Further, although certain aspects of observational EWAS, such as propensity for confounding, will always hinder interpretation of results, other aspects of this thesis may not apply to data from other tissues. 

There is overwhelming evidence that complex traits are polygenic (influenced by multiple genes) [@Timpson2017; @Visscher2017] and some hypothesise that they may even be omnigenic (influenced by potentially all genes, even if the influence of some genes is almost immeasurably small) [@Boyle2017]. As DNA methylation is highly correlated with factors of gene expression regulation, it seems likely that either DNA methylation or something coupled to it has a substantial impact on the aetiology of many complex traits. Thus, the issues uncovered with EWAS in this thesis, such as the lack of covariation between DNA methylation and complex traits, may be because too few sites are being measured in the wrong tissues. Finucane et al. devised a method that used LD-score regression and publically available gene expression data to identify the most relevant tissue and cell types for 34 traits [@Finucane2018]. Without access to large datasets with DNA methylation available in a plethora of tissues, molecular epidemiologists could rely on inferences made from studies such as this to help design future EWAS. Importantly, before such work is used to inform future studies, the definition of the relevant cell type should be clear. This will undoubtedly depend on the phenotype of interest, but will also differ based on the hypotheses being tested.

The impact of different methylation states of the genome, such as hydroxymethylation, is also beginning to be explored [@Wilkins2020; @Marshall2020; @Greco2016]. The current methods employed by EWAS, which revolve around bisulphite conversion, capture all methylation states as one measurement. If these states have differing effects, separating these out will increase power and impart greater biological understanding. 

In a similar vein to other omics, sequencing technology will likely continue to improve and will become cheap enough to use across population samples [@Shendure2017; @Stark2019]. Therefore, identifying the differential methylation states using techniques such as single-molecule real-time (SMRT) sequencing will hopefully become more frequent [@Song2012]. Also, despite the vast majority of EWAS being conducted using blood (__Table \@ref(tab:study-data-tab)__), there are `r study_dat[study_dat[["study-trait"]] == "Number of tissue types", "value", drop=T]` tissue types recorded in The EWAS Catalog. The Cancer Genome Atlas (TCGA) has recorded DNA methylation data in a large number of tissues also [@Weinstein2013], and the trend to measure other tissues is likely to continue, especially as the number of reliable sites identified in EWAS tends to be low. As discussed in __Section \@ref(appeal-of-dnam)__, DNA methylation is a relatively stable molecular trait and this is consistent across tissues. Therefore, if tissues can be sampled, measurement of DNA methylation is simpler than other molecular traits such as RNA levels and histone modifications across any tissue. In addition, manuscripts using techniques to measure gene expression and DNA methylation in single cells are becoming more common [@Hedlund2018; @Papalexi2018; @Karemaker2018; @Gravina2015; @Farlik2015]. Simple experiments have also shown that single cell RNA sequencing can give much greater resolution of cell types in a sample compared to other techniques [@Lappalainen2017]. Given the issues of cell specificity, these studies may prove pivotal in advancement of the field of epigenetic epidemiology. 

As discussed in __Chapter \@ref(properties-of-ewas)__, DNA methylation is a binary feature, yet by pooling groups of cells and measuring methylation at each site as a proportion of the DNA molecules that are methylated, a continuous measure is derived. If cells of the same type are identical then one might expect they would be methylated at the same sites as one another [@Birney2016]. However, a mosaicism of methylation across cells may exist. If so, then pooling groups of cells for analysis, and taking account of differences between higher order cell types, will remain a valid approach for EWAS. However, if strong evidence is provided that this cellular mosaicism is improbable between cells of the same type, rapid movement towards single-cell techniques is likely.

### New methods and data {#new-methods-and-data}

There is still substantial interest in assessing the association between DNA methylation and complex traits. In 2020, there have been over 100 EWAS conducted that have been or will be added to The EWAS Catalog. These EWAS are excluding studies that did not meet the criteria for inclusion into the database and studies that did not report findings relevant to the database. Further, new methods are being developed to measure the association between DNA methylation and complex traits. Two recent examples include Zhang F et al., who created a command line tool to conduct EWAS using different models, OSCA, [@Zhang2019] and a study that developed a Bayesian approach to conduct EWAS, using various confounders including genotype in their model [@TrejoBanos2020]. 

This interest will hopefully lead to a continuation in developments across the field of molecular epidemiology, but the complex nature of molecular pathways to disease may limit the speed at which robust aetiological inferences are made. Causal inference methods such as MR aid aetiological studies, but the lack of independent instruments coupled with the complexity of gene regulation make it difficult to be confident the key assumptions of MR (__Figure \@ref(fig:mr-diagram)__) are met [@Relton2010; @Richardson2018; @Relton2012; @Min2020]. Colocalization is now being readily used in the context of molecular MR [@Min2020; @Richardson2018], but this only provides evidence that the complex trait and the molecular phenotype have the same putatively causal genetic variant(s) [@Richardson2018], and do not provide evidence against these variants being pleiotropic. 

Lack of large-scale studies was a problem in identifying valid instruments for individual CpG sites, including within my study in __Chapter \@ref(dnam-lung-cancer-mr)__, but this has been partially eleviated by the formation of Genetics of DNA Methylation Consortium (GoDMC) [@Min2020]. This consortium has already yielded interesting results. Min et al. performed MR to establish whether there was likely an effect of DNA methylation changes, measured by the HM450, on 116 complex traits [@Min2020]. Similarly to the results presented in this thesis, they found that evidence for the aetiological relevance of DNA methylation measured across those sites was lacking. In the future, establishing tissue-specific mQTLs will be important in understanding whether tissue-specific DNA methylation changes are likely to have an effect on complex trait variation.

For prediction, the complexity of establishing causality and understanding cellular function need not apply. Whether the goal is to identify DNA methylation differences that co-exist with disease states and could therefore help diagnose these states, or truly predict diseases before they arrive is important to consider. For the former, cross-sectional studies suffice, but for the latter cohort studies that have DNA methylation data collected prior to disease onset such as ALSPAC [@Fraser2013; @Boyd2013] are required. Larger studies, such as Generation Scotland [@Smith2013], provide the opportunity for developing more precise predictors and further, the power gained from the extra samples would enable one to estimate the total predictive capacity of DNA methylation for individual traits with relatively high precision using the methods laid out in __Chapter \@ref(h2ewas-chapter)__.

### Beyond DNA methylation {#beyond-dnam}

As the premise of aetiological EWAS is that DNA methylation changes may impact upon some cellular function, which leads to phenotypic changes, reliable associations between DNA methylation and complex traits is not enough to impart translatable opportunities. How DNA methylation is changing these cellular functions is key for two reasons, 1. It might not be possible to target DNA methylation directly to treat/prevent disease and 2. DNA methylation may just be tagging some other epigenetic mark and in fact may be inconsequential to phenotypic state. Therefore, once a link between DNA methylation and complex traits is established, it is required that experimental work is undertaken to fill in the gap of how DNA methylation changes relate to cellular function differences that influence the trait.

With initiatives such as ENCODE and the Roadmap Epigenomics Project [@Dunham2012; @RoadmapEpigenomicsConsortium2015], along with gene expression datasets such as GTEx [@GTExConsortium2013], the functional implications of DNA methylation changes may be predicted. However, experimental work should be used to confirm these predictions.

As discussed in __Section \@ref(dnam-as-part-of-regulation)__, DNA methylation is not the only epigenetic mark in human cells. Technology has been a hinderance to population based studies of histone modifications genome-wide. At least one such study has been conducted [@Marzi2018], and the frequency is likely to increase as the cost of high throughput chromatin immunoprecipitation procedures, such as ChIP-seq, decreases.  

Recently large-scale datasets have been developed to analyse the relationship between protein expression and gene expression with complex traits [@GTExConsortium2013; @Uhlen2015; @Thul2017]. Proteins are often the ultimate target for many pharmaceuticals. As understanding of function and quantification of proteins and their variants increases, protein-based epidemiological studies may supersede EWAS with regards to their frequency. Indeed pharmaceutical companies may start interogating potential drugs by identifying targetable proteins. DNA methylation, and other molecular traits, may also add to the evidence that regulation of a particular protein target is important for disease development or progression. However, as discussed, the interpretability of EWAS makes discerning whether or not observed differences in DNA methylation between individuals are important for disease aetiology.

Similarly to DNA methylation studies, problems with confounding and cell specificity will be present in protein and gene based studies. Modifications to RNA transcripts and proteins provide an additional level of complication. RNA transcripts can be spliced and altered by post-transcriptional modifications and proteins can be modified post-translation of gene transcript [@Hafner2019; @Corbett2018; @Wang2014; @Filipowicz2008]. In extreme cases, proteins have been known to have their function reversed after their state is modified. The protein p53 naturally acts as a tumour suppressor, but during cancer development there is evidence that the protein can be modified and become oncogenic [@Hafner2019]. 

Studying each molecular measure will come with its own difficulties, and each can add to our understanding of human phenotypic variation. Therefore, instead of any one study superseding others, it seems likely that using a combination to untangle biological complexity will become more common. In fact, one thing Lapallainen and Greally stress is how, in their opinion, investment in deep molecular phenotyping of the same samples, although expensive, will likely pay high dividends with regards to interpretability of molecular epidemiological studies [@Lappalainen2017]. 

## Final conclusions {#final-conclusions}

The human body is unfathomably complex. Understanding it requires a huge concerted effort from the entire, global research community. Returning to the example of _p53_, by 2010 there had been nearly 50,000 studies on this single family of genes [@Lane2010]. The decades of research conducted to elucidate the function, importance and interactions of this single family of genes illustrates the patience required to understand the molecular underpinnings of disease. Given the brevity of existance molecular epidemiology has had, I'd argue that judging whether it has been a success or failure is a frivolous task. However, to speed up development of the field, there is a need to understand the limits of the current methodologies.

This thesis provides evidence that the study design of current EWAS is unlikely to provide substantial improvements in the ability to predict or understand the aetiological aspects of complex traits. 

Key next steps will be diversifying the tissue and cell types collected as well as the DNA methylation sites measured in the genome. Further, more experimental studies, that will inevitably come with time, will aid in epidemiological inferences.

It is important to note that this thesis, and many other molecular epidemiology studies, primarily focuses on samples of European origin. Datasets exist that contain individuals from different ethnic backgrounds and there have been EWAS conducted in non-Europeans (__Table \@ref(tab:study-data-tab)__), but there is certainly a bias towards European samples. To realise the full potential of molecular epidemiology and to help benefit all in our society, efforts should be made to reduce the bias in sample collection and expand analyses, where possible, to those of all ethnic backgrounds. 

Despite the apparent shortcomings, there is still great potential for EWAS to uncover important facets of complex trait biology. Imperative to the development of the studies will be the constant re-assessment of the predictive capacity and aetiological insights that can be gleaned from DNA methylation as more data is collected in different samples.