<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Appraising the causal relevance of DNA methylation for risk of lung cancer | thesis.utf8</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.18.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Appraising the causal relevance of DNA methylation for risk of lung cancer | thesis.utf8" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Appraising the causal relevance of DNA methylation for risk of lung cancer | thesis.utf8" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="6-ewas-gwas-comp-chapter.html"/>
<link rel="next" href="conclusion.html"/>
<script src="libs/header-attrs-2.1.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#dna-methylation-as-part-of-the-regulatory-machinery"><i class="fa fa-check"></i><b>1.1</b> DNA methylation as part of the regulatory machinery</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#gene-expression-regulation"><i class="fa fa-check"></i><b>1.1.1</b> Gene expression regulation</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#defining-epigenetics"><i class="fa fa-check"></i><b>1.1.2</b> Defining epigenetics</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#histone-modifications"><i class="fa fa-check"></i><b>1.1.3</b> Histone modifications</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-introduction.html"><a href="1-introduction.html#dna-methylation"><i class="fa fa-check"></i><b>1.1.4</b> DNA methylation</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#dnam-phs"><i class="fa fa-check"></i><b>1.2</b> Use of DNA methylation in a population setting </a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="1-introduction.html"><a href="1-introduction.html#appeal-of-dnam"><i class="fa fa-check"></i><b>1.2.1</b> The appeal of DNA methylation to epidemiologists</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-introduction.html"><a href="1-introduction.html#ewas"><i class="fa fa-check"></i><b>1.2.2</b> Epigenome-wide association studies</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-introduction.html"><a href="1-introduction.html#problems-for-ewas"><i class="fa fa-check"></i><b>1.2.3</b> Problems for EWAS</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#genetics-in-ewas"><i class="fa fa-check"></i><b>1.3</b> Using methods from genetics to help inform future EWAS</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#gwas-catalog"><i class="fa fa-check"></i><b>1.3.1</b> The GWAS Catalog</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#heritability"><i class="fa fa-check"></i><b>1.3.2</b> Total variance captured by all sites measured genome-wide</a></li>
<li class="chapter" data-level="1.3.3" data-path="1-introduction.html"><a href="1-introduction.html#inferring-biology-from-signals"><i class="fa fa-check"></i><b>1.3.3</b> Inferring biology from signals</a></li>
<li class="chapter" data-level="1.3.4" data-path="1-introduction.html"><a href="1-introduction.html#establishing-causality"><i class="fa fa-check"></i><b>1.3.4</b> Establishing causality</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#overview-of-thesis-aims"><i class="fa fa-check"></i><b>1.4</b> Overview of thesis aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-methods.html"><a href="2-methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a></li>
<li class="chapter" data-level="3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html"><i class="fa fa-check"></i><b>3</b> The EWAS Catalog: a database of epigenome-wide association studies</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#implementation"><i class="fa fa-check"></i><b>3.3.1</b> Implementation</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#overview-of-publication-data-extraction"><i class="fa fa-check"></i><b>3.3.2</b> Overview of publication data extraction</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#overview-of-geo-data-extraction"><i class="fa fa-check"></i><b>3.3.3</b> Overview of GEO data extraction</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#ewas-methods"><i class="fa fa-check"></i><b>3.3.4</b> EWAS methods</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#database-interface-and-use"><i class="fa fa-check"></i><b>3.4.1</b> Database interface and use</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-properties-of-ewas.html"><a href="4-properties-of-ewas.html"><i class="fa fa-check"></i><b>4</b> Properties of EWAS</a></li>
<li class="chapter" data-level="5" data-path="5-h2ewas-chapter.html"><a href="5-h2ewas-chapter.html"><i class="fa fa-check"></i><b>5</b> h2ewas</a></li>
<li class="chapter" data-level="6" data-path="6-ewas-gwas-comp-chapter.html"><a href="6-ewas-gwas-comp-chapter.html"><i class="fa fa-check"></i><b>6</b> EWAS-GWAS comparison</a></li>
<li class="chapter" data-level="7" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html"><i class="fa fa-check"></i><b>7</b> Appraising the causal relevance of DNA methylation for risk of lung cancer</a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#abstract-1"><i class="fa fa-check"></i><b>7.1</b> Abstract</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#background"><i class="fa fa-check"></i><b>7.1.1</b> Background</a></li>
<li class="chapter" data-level="7.1.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#methods-2"><i class="fa fa-check"></i><b>7.1.2</b> Methods</a></li>
<li class="chapter" data-level="7.1.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#results-1"><i class="fa fa-check"></i><b>7.1.3</b> Results</a></li>
<li class="chapter" data-level="7.1.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#conclusions"><i class="fa fa-check"></i><b>7.1.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#introduction-2"><i class="fa fa-check"></i><b>7.2</b> Introduction</a></li>
<li class="chapter" data-level="7.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#methods-3"><i class="fa fa-check"></i><b>7.3</b> Methods</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-study-details"><i class="fa fa-check"></i><b>7.3.1</b> EWAS study details</a></li>
<li class="chapter" data-level="7.3.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-meta-analysis"><i class="fa fa-check"></i><b>7.3.2</b> EWAS Meta-analysis</a></li>
<li class="chapter" data-level="7.3.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#mendelian-randomization-1"><i class="fa fa-check"></i><b>7.3.3</b> Mendelian randomization</a></li>
<li class="chapter" data-level="7.3.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#sample-2-transdisciplinary-research-in-cancer-of-the-lung-and-the-international-lung-cancer-consortium"><i class="fa fa-check"></i><b>7.3.4</b> Sample 2: Transdisciplinary Research in Cancer of the Lung and The International Lung Cancer Consortium</a></li>
<li class="chapter" data-level="7.3.5" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#supplementary-analyses"><i class="fa fa-check"></i><b>7.3.5</b> Supplementary analyses</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#results-2"><i class="fa fa-check"></i><b>7.4</b> Results</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-meta-analysis-1"><i class="fa fa-check"></i><b>7.4.1</b> EWAS meta-analysis</a></li>
<li class="chapter" data-level="7.4.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#mendelian-randomization-2"><i class="fa fa-check"></i><b>7.4.2</b> Mendelian randomization</a></li>
<li class="chapter" data-level="7.4.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#tumour-and-adjacent-normal-lung-tissue-methylation-patterns"><i class="fa fa-check"></i><b>7.4.3</b> Tumour and adjacent normal lung tissue methylation patterns</a></li>
<li class="chapter" data-level="7.4.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#gene-expression-associated-with-mqtls-in-blood-and-lung-tissue"><i class="fa fa-check"></i><b>7.4.4</b> Gene expression associated with mQTLs in blood and lung tissue</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#discussion-1"><i class="fa fa-check"></i><b>7.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="B" data-path="B-the-second-appendix-for-fun.html"><a href="B-the-second-appendix-for-fun.html"><i class="fa fa-check"></i><b>B</b> The Second Appendix, for Fun</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dnam-lung-cancer-mr" class="section level1" number="7">
<h1><span class="header-section-number">Chapter 7</span> Appraising the causal relevance of DNA methylation for risk of lung cancer</h1>
<div id="abstract-1" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Abstract</h2>
<div id="background" class="section level3" number="7.1.1">
<h3><span class="header-section-number">7.1.1</span> Background</h3>
<p>DNA methylation changes in peripheral blood have recently been identified in relation to lung cancer risk. Some of these changes have been suggested to mediate part of the effect of smoking on lung cancer. However, limitations with conventional mediation analyses mean that the causal nature of these methylation changes has yet to be fully elucidated.</p>
</div>
<div id="methods-2" class="section level3" number="7.1.2">
<h3><span class="header-section-number">7.1.2</span> Methods</h3>
<p>We first performed a meta-analysis of four epigenome-wide association studies (EWAS) of lung cancer (918 cases, 918 controls). Next, we conducted a two-sample Mendelian randomization analysis, using genetic instruments for methylation at CpG sites identified in the EWAS meta-analysis, and 29,863 cases and 55,586 controls from the TRICL-ILCCO lung cancer consortium, to appraise the possible causal role of methylation at these sites on lung cancer.</p>
</div>
<div id="results-1" class="section level3" number="7.1.3">
<h3><span class="header-section-number">7.1.3</span> Results</h3>
<p>16 CpG sites were identified from the EWAS meta-analysis (FDR &lt; 0.05), 14 of which we could identify genetic instruments for. Mendelian randomization provided little evidence that DNA methylation in peripheral blood at the 14 CpG sites play a causal role in lung cancer development (FDR &gt; 0.05), including for cg05575921-<em>AHRR</em> where methylation is strongly associated with both smoke exposure and lung cancer risk.</p>
</div>
<div id="conclusions" class="section level3" number="7.1.4">
<h3><span class="header-section-number">7.1.4</span> Conclusions</h3>
<p>The results contrast with previous observational and mediation analysis, which have made strong claims regarding the causal role of DNA methylation. Thus, previous suggestions of a mediating role of methylation at sites identified in peripheral blood, such as cg05575921-<em>AHRR</em>, could be unfounded. However, this study does not preclude the possibility that differential DNA methylation at other sites is causally involved in lung cancer development, especially within lung tissue.</p>
</div>
</div>
<div id="introduction-2" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Introduction</h2>
<p>Lung cancer is the most common cause of cancer-related death worldwide <span class="citation">(<span class="citeproc-not-found" data-reference-id="Ferlay2013"><strong>???</strong></span>)</span>. Several DNA methylation changes have been recently identified in relation to lung cancer risk <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fasanelli2015"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="McCarthy2016"><strong>???</strong></span>)</span>. Given the plasticity of epigenetic markers, any DNA methylation changes that are causally linked to lung cancer are potentially appealing targets for intervention <span class="citation">(<span class="citeproc-not-found" data-reference-id="Strathdee2002"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Jones2002"><strong>???</strong></span>)</span>. However, these epigenetic markers are sensitive to reverse causation, being affected by cancer processes <span class="citation">(<span class="citeproc-not-found" data-reference-id="Jones2002"><strong>???</strong></span>)</span>, and are also prone to confounding, for example by socio-economic and lifestyle factors [<span class="citation">(<span class="citeproc-not-found" data-reference-id="Borghol2012"><strong>???</strong></span>)</span>; Elliott2014].</p>
<p>One CpG site, cg05575921 within the aryl hydrocarbon receptor repressor (<em>AHRR</em>) gene, has been consistently replicated in relation to both smoking <span class="citation">(<span class="citeproc-not-found" data-reference-id="Joehanes2016"><strong>???</strong></span>)</span> and lung cancer <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fasanelli2015"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Bojesen2017"><strong>???</strong></span>)</span> and functional evidence suggests that this region could be causally involved in lung cancer <span class="citation">(<span class="citeproc-not-found" data-reference-id="Zudaire2008"><strong>???</strong></span>)</span>. However, the observed association between methylation and lung cancer might simply reflect separate effects of smoking on lung cancer and DNA methylation, i.e. the association may be a result of confounding <span class="citation">(<span class="citeproc-not-found" data-reference-id="Richmond2016"><strong>???</strong></span>)</span>, including residual confounding after adjustment for self-reported smoking behaviour <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fewell2007"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Munafo2012"><strong>???</strong></span>)</span>. Furthermore, recent epigenome-wide association studies (EWAS) for lung cancer have revealed additional CpG sites which may be causally implicated in development of the disease <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fasanelli2015"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span>.</p>
<p>Mendelian randomization (MR) uses genetic variants associated with modifiable factors as instruments to infer causality between the modifiable factor and outcome, overcoming most unmeasured or residual confounding and reverse causation <span class="citation">(<span class="citeproc-not-found" data-reference-id="DaveySmith2014"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="DaveySmith2003"><strong>???</strong></span>)</span>. In order to infer causality, three core assumptions of MR should be met: 1) The instrument is associated with the exposure, 2) The instrument is not associated with any confounders, 3) The instrument is associated with the outcome only through the exposure. MR may be adapted to the setting of DNA methylation <span class="citation">(<span class="citeproc-not-found" data-reference-id="Relton2012"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Relton2015"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Richardson2017"><strong>???</strong></span>)</span> with the use of single nucleotide polymorphisms (SNPs) that correlate with methylation of CpG sites, known as methylation quantitative trait loci (mQTLs) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Gaunt2016"><strong>???</strong></span>)</span>.</p>
<p>In this study, we performed a meta-analysis of four lung cancer EWAS (918 case-control pairs) from prospective cohort studies to identify CpG sites associated with lung cancer risk and applied MR to investigate whether the observed DNA methylation changes at these sites are causally linked to lung cancer.</p>
<p>In this chapter, Dr Rebecca Richmond performed analysis in the CCHS cohort (see section <a href="7-dnam-lung-cancer-mr.html#potential-causal-effect-of-ahrr-methylation-on-lung-cancer-risk-one-sample-mr">Potential causal effect of <em>AHRR</em> methylation on lung cancer risk: one sample MR</a> of the results) and contributed to writing the introduction and discussion. I completed all the analysis and wrote each section except the methods and results sections for which I did not complete the analysis.</p>
</div>
<div id="methods-3" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Methods</h2>
<div id="ewas-study-details" class="section level3" number="7.3.1">
<h3><span class="header-section-number">7.3.1</span> EWAS study details</h3>
<p>A meta-analysis of four lung cancer case-control EWAS was conducted to identify DNA methlyation sites associated with lung cancer. DNA methylation in each EWAS was assessed using the Illumina Infinium® HumanMethylation450 BeadChip. All EWAS are nested within prospective cohorts that measured DNA methylation in peripheral blood samples before diagnosis: EPIC-Italy (185 case-control pairs), Melbourne Collaborative Cohort Study (MCCS) (367 case-control pairs), Norwegian Women and Cancer (NOWAC) (132 case-control pairs) and the Northern Sweden Health and Disease Study (NSHDS) (234 case-control pairs). Study populations, laboratory methods, data pre-processing and quality control methods have been described in detail elsewhere <span class="citation">(<span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span> 3 and are outlined below.</p>
<p>At the various laboratory sites, samples were distributed into 96-well plates and processed in chips of 12 arrays (8 chips per plate) with case-control pairs arranged randomly on the same chip. Methylation data were pre-processed and normalized in each study, and probe filtering was performed as previously described <span class="citation">(<span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span>, leaving 465,886 CpGs suitable for the analysis in EPIC-Italy, 485,330 CpGs in MCCS, 450,890 CpGs in NOWAC and 482,867 CpGs in NSHDS.</p>
<div id="european-prospective-investigation-into-cancer-and-nutrition-italy-epic-italy" class="section level4" number="7.3.1.1">
<h4><span class="header-section-number">7.3.1.1</span> European Prospective Investigation into Cancer and Nutrition-Italy (EPIC-Italy)</h4>
<p>EPIC-Italy includes 47,749 volunteers (32,579 women) aged 35–70 years at the time of recruitment (1992–1998). Anthropometric measurements and lifestyle variables including detailed information on smoking history were collected at recruitment through standardized questionnaires, together with a blood sample. Within EPIC-Italy we conducted a nested case-control study utilizing incident cases diagnosed within follow-up and healthy controls individually matched to cases by gender, date of birth (±5 years), date of inclusion in the study and study centre. Analysis was performed for 185 incident cases diagnosed within follow-up and matched controls. Laboratory procedures were carried out at the Human Genetics Foundation (Turin, Italy) and DNA extracted from buffy coats as previously described <span class="citation">(<span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span>. All participants signed an informed consent form, and the ethical review boards of the International Agency for Research on Cancer and of each local participating centre approved the study protocol.</p>
</div>
<div id="melbourne-collaborative-cohort-study-mccs" class="section level4" number="7.3.1.2">
<h4><span class="header-section-number">7.3.1.2</span> Melbourne Collaborative Cohort Study (MCCS)</h4>
<p>The MCCS is a prospective cohort study of 41,514 volunteers (24,469 women) aged between 27 and 76 years at baseline (1990-1994). At baseline attendance, participants completed questionnaires that measured demographic characteristics and lifestyle factors. Height and weight were directly measured, and a blood sample was collected and stored. Incident cases of lung cancer were identified through linkage with the State and National Cancer Registries during follow-up up to the end of 2011. The MCCS sample included 367 cases and 367 matched controls selected from MCCS participants who were lung cancer free at the age of diagnosis of the matching case (density sampling). Matching variables included gender, date of blood collection (within 6 months), date of birth (within 1 year), country of birth (Australia and UK versus Southern Europe), type of biospecimen (lymphocyte, buffy coat and dried blood spot) and smoking status (never smokers; short-term former smokers: quitting smoking less than 10 years before blood draw; long-term former smokers: quitting smoking 10 years or more before blood draw; current light smokers: less than 15 cigarettes per day at blood draw; and current heavy smokers: 15 cigarettes or more at blood draw). For the MCCS, laboratory procedures were carried out at the Genetic Epidemiology Laboratory, the University of Melbourne according to manufacturers’ protocols. DNA extraction from lymphocytes and buffy coats was performed as previously described <span class="citation">(<span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span>. The Cancer Council Victoria’s Human Research Ethics Committee approved the study protocol. Subjects gave written consent to participate and for the investigators to obtain access to their medical records.</p>
</div>
<div id="norwegian-women-and-cancer-nowac" class="section level4" number="7.3.1.3">
<h4><span class="header-section-number">7.3.1.3</span> Norwegian Women and Cancer (NOWAC)</h4>
<p>The biobank of the NOWAC cohort was established in the years 2003-2006. Those who filled in an eight-page questionnaire and accepted the invitation to donate blood were sent blood drawing equipment together with a two-page epidemiological questionnaire. Around 50 000 women returned two tubes of blood to the Institute of Community Medicine at UiT The Arctic University of Norway and data linkage to the National Cancer Registry of Norway was performed. During follow-up to the end of 2011, 132 eligible cases of lung cancer were identified and were used for the EWAS. For each case, one control with an available blood sample was selected and matched on time since blood sampling and year of birth in order to control for effects of storage time and ageing. The cases and the controls were processed together for all laboratory procedures in order to reduce any batch effect. Laboratory procedures were carried out at the Human Genetics Foundation (Turin, Italy). DNA extraction from buffy coats was performed as previously described <span class="citation">(<span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span>. All participants gave informed consent. The study was approved by the Regional Committee for Medical and Health Research Ethics in North Norway. Data storage and linkage was approved by the Norwegian Data Inspectorate.</p>
</div>
<div id="northern-sweden-health-and-disease-study-nshds" class="section level4" number="7.3.1.4">
<h4><span class="header-section-number">7.3.1.4</span> Northern Sweden Health and Disease Study (NSHDS)</h4>
<p>NSHDS is an ongoing prospective cohort and intervention study intended for health promotion of the population of Västerbotten County in northern Sweden. All residents were invited to participate by attending a health check-up at their local health care centre at 40, 50 and 60 years of age. At the health check-up, participants were asked to complete a self-administered questionnaire covering various factors such as education, smoking habits, physical activity and diet. In addition, height and weight were measured and participants were asked to donate a blood sample. Incident lung cancer cases were identified through linkage to the regional cancer registry. One control was chosen at random for each lung cancer case from appropriate risk sets consisting of all cohort members alive and free of cancer (except non-melanoma skin cancer) at the time of diagnosis of the index case. Matching criteria were the same as for the MCCS except there was no matching for type of biospecimens as DNA was extracted from whole blood for all samples. After quality control, a total of 234 incident lung cancer cases and 234 individually matched controls were available for this analysis. Laboratory procedures for NSHDS were carried out at two sites. DNA extraction from the buffy coat was conducted at Umeå University, Sweden, as previously described. Illumina Infinium HumanMethylation450 BeadChip analysis was conducted at the ALSPAC/IEU Laboratory at the University of Bristol. All study subjects provided written informed consent at time of the recruitment into the NSHDS.</p>
</div>
</div>
<div id="ewas-meta-analysis" class="section level3" number="7.3.2">
<h3><span class="header-section-number">7.3.2</span> EWAS Meta-analysis</h3>
<p>To quantify the association between the methylation level at each CpG and the risk of lung cancer conditional logistic regression models were fitted for beta values of methylation (which ranges from 0 (no cytosines methylated) to 1 (all cytosines methylated)) on lung cancer status for the four studies. The cases and controls in each study were matched, details can be found above. Surrogate variables were computed in the four studies using the SVA R package <span class="citation">(<span class="citeproc-not-found" data-reference-id="Leek2016"><strong>???</strong></span>)</span> and the proportion of CD8+ and CD4+ T cells, B cells, monocytes, natural killer cells and granulocytes within whole blood were derived from DNA methylation <span class="citation">(<span class="citeproc-not-found" data-reference-id="Houseman2012"><strong>???</strong></span>)</span>. The following EWAS models were included in the meta-analysis: Model 1 – unadjusted; Model 2 – adjusted for 10 surrogate variables (SVs); Model 3 – adjusted for 10 SVs and derived cell proportions. EWAS stratified by smoking status was also conducted (never (N=304), former (N=648) and current smoking (N=857)). For Model 1 and Model 2, the case-control studies not matched on smoking status (EPIC-Italy and NOWAC) were adjusted for smoking.</p>
<p>An inverse-variance weighted fixed effects meta-analysis was performed of the EWAS (918 case-control pairs) using the <a href="http://csg.sph.umich.edu/abecasis/metal/">METAL software</a>. Direction of effect, effect estimates and the I<sup>2</sup> statistic were used to assess heterogeneity across the studies in addition to effect estimates across smoking strata (never, former and current). All sites identified at a false discovery rate (FDR) &lt; 0.05 in Model 2 and 3 were also present in the sites identified in Model 1. The effect size differences between models for all sites identified in Model 1 were assessed by a Kruskal-Wallis test and a post-hoc Dunn’s test. There was little evidence for a difference (P &gt; 0.1), so to maximize inclusion into the MR analyses we took forward the sites identified in the unadjusted model (Model 1).</p>
</div>
<div id="mendelian-randomization-1" class="section level3" number="7.3.3">
<h3><span class="header-section-number">7.3.3</span> Mendelian randomization</h3>
<p>Two-sample MR was used to establish potential causal effect of differential methylation on lung cancer risk <span class="citation">(<span class="citeproc-not-found" data-reference-id="Inoue2010"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Pierce2013"><strong>???</strong></span>)</span>.</p>
<div id="sample-1-accessible-resource-for-integrated-epigenomic-studies-aries" class="section level4" number="7.3.3.1">
<h4><span class="header-section-number">7.3.3.1</span> Sample 1: Accessible Resource for Integrated Epigenomic Studies (ARIES)</h4>
<p>In the first sample, mQTL-methylation effect estimates (<span class="math inline">\(\beta_{GP}\)</span>) for each CpG site of interest were identified in an mQTL database from the Accessible Resource for Integrated Epigenomic Studies (ARIES) (<a href="http://www.mqtldb.org" class="uri">http://www.mqtldb.org</a>). Details on the methylation pre-processing, genotyping and quality control (QC) pipelines are outlined below.</p>
<p><strong>DNA methylation data</strong>
Samples were drawn from the Avon Longitudinal Study of Parents and Children [<span class="citation">(<span class="citeproc-not-found" data-reference-id="Boyd2013"><strong>???</strong></span>)</span>; Fraser2013]. Blood from 1,018 mother–child pairs were selected for analysis as part of the Accessible Resource for Integrative Epigenomic Studies (ARIES, <a href="http://www.ariesepigenomics.org.uk/" class="uri">http://www.ariesepigenomics.org.uk/</a>) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Relton2015"><strong>???</strong></span>)</span>. There are three timepoints in children and two in their mothers, the timepoints with mean ages (in brackets) in ARIES are as follows for children: birth, childhood (7.5), adolescence (17.1) and for mothers: during pregnancy (28.7), and at middle age (46.9). Following DNA extraction, samples were bisulphite converted using the Zymo EZ DNA Methylation<sup>TM</sup> kit (Zymo, Irvine, CA, USA). Following conversion, genome-wide methylation was measured using the Illumina Infinium HumanMethylation450 (HM450) BeadChip. Methylation data were normalised in R with the watermelon package <span class="citation">(<span class="citeproc-not-found" data-reference-id="Pidsley2013"><strong>???</strong></span>)</span> using the Touleimat and Tost <span class="citation">(<span class="citeproc-not-found" data-reference-id="Touleimat2012"><strong>???</strong></span>)</span> algorithm to reduce the non-biological differences between probes. Methylation data in ARIES were rank-normalised to remove outliers, and then Matrix eQTL software <span class="citation">(<span class="citeproc-not-found" data-reference-id="Gaunt2016"><strong>???</strong></span>)</span> was used to perform preliminary association analysis of SNPs with all CpG sites in the Illumina Infinium HM450 array with the exception of those failing QC, and those reported to map to more than one location (n=19,834) or to contain a genetic variant at the CpG site (n=74,182) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Naeem2014"><strong>???</strong></span>)</span>.</p>
<p><strong>Genetic data</strong>
Children were genotyped using the Illumina HumanHap550 quad genome-wide SNP genotyping platform (Illumina Inc., San Diego, USA) by the Wellcome Trust Sanger Institute (WTSI, Cambridge, UK) and the Laboratory Corporation of America (LCA, Burlington, NC, USA). Individuals were excluded on the basis of incorrect gender assignment; abnormal heterozygosity (&lt;0.320 or &gt;0.345 for WTSI data; &lt;0.310 or &gt;0.330 for LCA data); high missingness (&gt;3%); cryptic relatedness (&gt;10% identity by descent) and non-European ancestry (detected by multidimensional scaling analysis). Following QC the final directly genotyped dataset contained 500,527 SNP loci.</p>
<p>Mothers were genotyped using the Illumina Human660W-quad genome-wide SNP genotyping platform (Illumina Inc., San Diego, USA) at the Centre National de Génotypage (CNG, Paris, France). Individuals were excluded based on non-European ancestry, missingness, relatedness, gender mismatches and heterozygosity. PLINK (v1.07) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Purcell2007"><strong>???</strong></span>)</span> was used to carry out quality control measures on an initial set of 10,015 subjects and 557,124 directly genotyped SNPs. Following QC the final directly genotyped dataset contained 526,688 SNP loci.</p>
<p>Imputation was performed to increase the SNP density for all genotyped mothers and children combined. Genotypes were phased together using ShapeIt, and then imputed against the 1000 genomes reference panel (phase 1 version 3, phased using ShapeIt version 2, December 2013, using all populations) using Impute (version 2.2.2). Genotypes were filtered to have Hardy-Weinberg equilibrium P &gt; 5x10<sup>-7</sup>, MAF &gt; 1% and imputation info score &gt; 0.8. Best guess genotypes were used for subsequent analysis. The final imputed dataset used for the analyses presented here contained 8,074,398 loci.</p>
<p>Written informed consent has been obtained from all ALSPAC participants. Ethical approval for the study was obtained from the ALSPAC Ethics and Law Committee and Local Research Ethics Committees.
Please note that the study website contains details of all the data that is available through a fully searchable data dictionary: <a href="http://www.bris.ac.uk/alspac/researchers/data-access/data-dictionary/" class="uri">http://www.bris.ac.uk/alspac/researchers/data-access/data-dictionary/</a></p>
</div>
</div>
<div id="sample-2-transdisciplinary-research-in-cancer-of-the-lung-and-the-international-lung-cancer-consortium" class="section level3" number="7.3.4">
<h3><span class="header-section-number">7.3.4</span> Sample 2: Transdisciplinary Research in Cancer of the Lung and The International Lung Cancer Consortium</h3>
<p>In the second sample, summary data was extracted from a GWAS meta-analysis of lung cancer risk conducted by the Transdisciplinary Research in Cancer of the Lung and The International Lung Cancer Consortium (TRICL-ILCCO) (29,863 cases, 55,586 controls) for individuals genotyped using the Illumina Infinium OncoArray-500K BeadChip (Illumina Inc. San Diego, CA) and independent samples for which prior genotyping was performed <span class="citation">(<span class="citeproc-not-found" data-reference-id="McKay2017"><strong>???</strong></span>)</span>. They were used to obtain mQTL-lung cancer estimates (<span class="math inline">\(\beta_{GD}\)</span>).</p>
<p>For each independent mQTL (r<sup>2</sup> &lt; 0.01), we calculated the log odds ratio (OR) per SD unit increase in methylation by the formula <span class="math inline">\(\frac{\beta_{GD}} {\beta_{GP}}\)</span> (Wald ratio). Standard errors were approximated by the delta method <span class="citation">(<span class="citeproc-not-found" data-reference-id="Thomas2007"><strong>???</strong></span>)</span>. Where multiple independent mQTLs were available for one CpG site, these were combined in a fixed effects meta-analysis after weighting each ratio estimate by the inverse variance of their associations with the outcome. Heterogeneity in Wald ratios across mQTLs was estimated using Cochran’s Q test, which can be used to indicate horizontal pleiotropy <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bowden2015"><strong>???</strong></span>)</span>. Differences between the observational and MR estimates were assessed using a Z-test for difference.</p>
<p>If there was evidence for an mQTL-CpG site association in ARIES in at least one time-point, it was assessed whether the mQTL replicated across time points in ARIES (FDR &lt; 0.05, same direction of effect). Further, this association was re-analysed using linear regression of methylation on each genotyped SNP available in an independent cohort (NSHDS), using rvtests <span class="citation">(<span class="citeproc-not-found" data-reference-id="Zhan2016"><strong>???</strong></span>)</span>. The same NSHDS samples on which DNA methylation was measured were genotyped using the Illumina Infinium OncoArray-500k BeadChip (Illumina Inc. San Diego, CA) and quality control parameters were applied under the recently published TRICL-ILCCO GWAS study on lung cancer <span class="citation">(<span class="citeproc-not-found" data-reference-id="McKay2017"><strong>???</strong></span>)</span>. Genetic imputation was performed on these samples using the Haplotype Reference Consortium (HRC) Panel (release 1) <span class="citation">(<span class="citeproc-not-found" data-reference-id="McCarthy2016"><strong>???</strong></span>)</span> through the Michigan Imputation Server <span class="citation">(<span class="citeproc-not-found" data-reference-id="Das2016"><strong>???</strong></span>)</span>. Replicated mQTLs were included where possible to reduce the effect of winner’s curse using effect estimates from ARIES. We assessed the instrument strength of the mQTLs by investigating the variance explained in methylation by each mQTL (r<sup>2</sup>) as well as the F-statistic in ARIES <strong>Supplementary Table 1</strong>. The power to detect the observational effect estimates in the two-sample MR analysis was assessed a priori, based on an alpha of 0.05, sample size of 29,863 cases and 55,586 controls (from TRICL-ILCCO) and calculated variance explained (r<sup>2</sup>).</p>
<p>MR analyses were also performed to investigate the impact of methylation on lung cancer subtypes in TRICL-ILCCO: adenocarcinoma (11,245 cases, 54,619 controls), small cell carcinoma (2791 cases, 20,580 controls), and squamous cell carcinoma (7704 cases, 54,763 controls). We also assessed the association in never smokers (2303 cases, 6995 controls) and ever smokers (23,848 cases, 16,605 controls) <span class="citation">(<span class="citeproc-not-found" data-reference-id="McKay2017"><strong>???</strong></span>)</span> 25. Differences between the smoking subgroups were assessed using a Z-test for difference.</p>
<p>We next investigated the extent to which the mQTLs at cancer-related CpGs were associated with four smoking behaviour traits which could confound the methylation-lung cancer association: number of cigarettes per day, smoking cessation rate, smoking initiation and age of smoking initiation using GWAS data from the Tobacco and Genetics (TAG) consortium (N=74,053) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Furberg2010"><strong>???</strong></span>)</span> 29.</p>
</div>
<div id="supplementary-analyses" class="section level3" number="7.3.5">
<h3><span class="header-section-number">7.3.5</span> Supplementary analyses</h3>
<div id="assessing-the-potential-causal-effect-of-ahrr-methylation-one-sample-mr" class="section level4" number="7.3.5.1">
<h4><span class="header-section-number">7.3.5.1</span> Assessing the potential causal effect of <em>AHRR</em> methylation: one sample MR</h4>
<p>Given previous findings implicating methylation at <em>AHRR</em> in relation to lung cancer <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fasanelli2015"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span> 2, 3, we performed a one-sample MR analysis <span class="citation">(<span class="citeproc-not-found" data-reference-id="Haycock2016"><strong>???</strong></span>)</span> 30 of <em>AHRR</em> methylation on lung cancer incidence using individual-level data from the Copenhagen City Heart Study (CCHS) (357 incident cases, 8401 remaining free of lung cancer). Copenhagen City Heart Study is a prospective study of the general population <span class="citation">(<span class="citeproc-not-found" data-reference-id="Kaur-Knudsen2011"><strong>???</strong></span>)</span>. Copenhagen residents were invited to complete a questionnaire and undergo a physical examination and are followed through a unique person identifier in the Danish health registries. All participants gave written informed consent, and a Danish ethics committee approved the study (KF100.2039/91).</p>
<p><strong>Phenotypic data</strong>
Participants were asked whether they smoked at the day of attendance or previously. If they answered affirmative to either of these questions, they were asked about their current and former smoking behaviour, including age of smoking initiation, age of smoking cessation, and number of daily consumed cigarettes, cheroots, cigars, and weekly grams of pipe tobacco. Based on these answers, participants were categorized as never, former, and current smokers. In addition, participants reported on alcohol consumption, occupational exposure to dust and/or welding fumes, exposure to passive smoking, education, and familial cases of lung cancer. The answers were reviewed together with an examiner at the day of attendance. Body mass index was calculated as measured weight in kilograms divided by measured height (in meters) squared.</p>
<p><strong>Methylation data</strong>
At the physical examination, blood samples were drawn for DNA from which <em>AHRR</em> methylation extent was measured <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bojesen2017"><strong>???</strong></span>)</span>. The <em>AHRR</em> cg05575921 methylation extent was measured in duplicate samples of bisulphite treated DNA from peripheral blood from 9,234 individuals. We used a Taqman assay developed in our own laboratory, and included standard curves, as well as internal controls in each 384-well plate. Coefficients of variation at the methylation level of 71% varied from 5.0 to 6.7%. Laboratory technicians were blinded to smoking and disease status of the individuals. Results were validated with pyrosequencing on a subset of samples.</p>
<p><strong>Genetic data</strong>
Genotypes from the iCOGs array <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bojesen2013"><strong>???</strong></span>)</span> and prospective data on lung cancer incidence were also available for these participants. Of the 9234 individuals, genotype data from iCOGS on 8778 were available. In short, DNA isolated from leukocytes was genotyped with a custom Illumina iSelect genotyping array, designed to test genetic variants related to breast, ovary and prostate cancer, comprising roughly 211,000 SNPs after rigorous quality control <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bojesen2013"><strong>???</strong></span>)</span>.</p>
<p><strong>Lung cancer data</strong>
For lung cancer (ICD7, codes 1624 or 4624 until 1977, and ICD10, code C34 from 1978 and onwards), the date of first diagnosis was taken from the national Danish Cancer Registry from 1943 to December 2012.</p>
<p><strong>Identification of mQTLs for CCHS one-sample MR</strong>
mQTLs located within 1Mb of cg05575921 <em>AHRR</em> were identified in ARIES (FDR&lt;0.05). Of those mQTLs which replicated within the CCHS, we performed an LD pruning step using a less stringent r<sup>2</sup> threshold of 0.2 and generated an unweighted allele score, calculated by coding and then summing the alleles to reflect the average number of methylation-increasing alleles carried by an individual. Associations between the allele score and several potential confounding factors (sex, alcohol consumption, smoking status, occupational exposure to dust and/or welding fumes, passive smoking) were investigated. Then MR analyses were performed using two-stage Cox regression, with adjustment for age and sex, and further stratified by smoking status.</p>
</div>
<div id="tumour-and-adjacent-normal-methylation-patterns" class="section level4" number="7.3.5.2">
<h4><span class="header-section-number">7.3.5.2</span> Tumour and adjacent normal methylation patterns</h4>
<p>DNA methylation data from lung cancer tissue and matched normal adjacent tissue (N=40 squamous cell carcinoma and N=29 adenocarcinoma), profiled as part of The Cancer Genome Atlas (TCGA), were used to assess tissue-specific DNA methylation changes across sites identified in the meta-analysis of EWAS, as outlined previously <span class="citation">(<span class="citeproc-not-found" data-reference-id="Teschendorff2015"><strong>???</strong></span>)</span> 31.</p>
</div>
<div id="mqtl-association-with-gene-expression" class="section level4" number="7.3.5.3">
<h4><span class="header-section-number">7.3.5.3</span> mQTL association with gene expression</h4>
<p>For the genes annotated to CpG sites identified in the lung cancer EWAS, we examined gene expression in whole blood and lung tissue using data from the gene-tissue expression (GTEx) consortium <span class="citation">(<span class="citeproc-not-found" data-reference-id="GTExConsortium2013"><strong>???</strong></span>)</span> 32.</p>
<p>Analyses were conducted in Stata (version 14) and R (version 3.2.2). For the two-sample MR analysis we used the MR-Base R package TwoSampleMR <span class="citation">(<span class="citeproc-not-found" data-reference-id="Hemani2016"><strong>???</strong></span>)</span> 33. An adjusted P value that limited the FDR was calculated using the Benjamini-Hochberg method <span class="citation">(<span class="citeproc-not-found" data-reference-id="Benjamini1995"><strong>???</strong></span>)</span> 34. All statistical tests were two-sided.</p>
</div>
</div>
</div>
<div id="results-2" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> Results</h2>
<p>A flowchart representing our study design along with a summary of our results at each step is displayed in Figure 1.</p>
<p>(Figure 1 here)</p>
<div id="ewas-meta-analysis-1" class="section level3" number="7.4.1">
<h3><span class="header-section-number">7.4.1</span> EWAS meta-analysis</h3>
<p>The basic meta-analysis adjusted for study-specific covariates identified 16 CpG sites which were hypomethylated in relation to lung cancer (FDR&lt;0.05, Model 1, Figure 2). Adjusting for 10 surrogate variables (Model 2) and derived cell counts (Model 3) gave similar results (Table 1). The direction of effect at the 16 sites did not vary between studies (median I2=38.6) (Supplementary Table 2), but there was evidence for heterogeneity of effect estimates at some sites when stratifying individuals by smoking status (Table 1).</p>
<p>(Figure 2 here)</p>
<p>(Table 1 here)</p>
</div>
<div id="mendelian-randomization-2" class="section level3" number="7.4.2">
<h3><span class="header-section-number">7.4.2</span> Mendelian randomization</h3>
<p>We identified 15 independent mQTLs (r2&lt;0.01) associated with methylation at 14 of 16 CpGs. Ten mQTLs replicated at FDR&lt;0.05 in NSHDS (Supplementary Table 3). MR power analyses indicated &gt;99% power to detect ORs for lung cancer of the same magnitude as those in the meta-analysis of EWAS.</p>
<p>There was little evidence for an effect of methylation at these 14 sites on lung cancer (FDR&gt;0.05, Supplementary Table 4). For nine of 14 CpG sites the point estimates from the MR analysis were in the same direction as in the EWAS, but of a much smaller magnitude (Z-test for difference, P&lt;0.001) (Figure 3).</p>
<p>For 9 of out the 16 mQTL-CpG associations, there was strong replication across time points (Supplementary Table 5) and 10 out of 16 mQTL-CpG associations replicated at FDR&lt;0.05 in an independent adult cohort (NSHDS). Using mQTL effect estimates from NSHDS for the 10 CpG sites that replicated (FDR&lt;0.05), findings were consistent with limited evidence for a causal effect of peripheral blood-derived DNA methylation on lung cancer (Supplementary Figure 1).</p>
<p>(Figure 3 here)</p>
<p>There was little evidence of different effect estimates between ever and never smokers at individual CpG sites (Supplementary Figure 2, Z-test for difference, P&gt;0.5). There was some evidence for a possible effect of methylation at cg21566642-ALPPL2 and cg23771366-PRSS23 on squamous cell lung cancer (OR=0.85 [95% confidence interval (CI)=0.75,0.97] and 0.91 [95% CI=0.84,1.00] per SD [14.4% and 5.8%] increase, respectively) as well as methylation at cg23387569-AGAP2, cg16823042-AGAP2, and cg01901332-ARRB1 on lung adenocarcinoma (OR=0.86 [95% CI=0.77,0.96], 0.84 [95% CI=0.74,0.95], and 0.89 [95% CI=0.80,1.00] per SD [9.47%, 8.35%, and 8.91%] increase, respectively). However, none of the results withstood multiple testing correction (FDR&lt;0.05) (Supplementary Figure 3). For those CpGs where multiple mQTLs were used as instruments (cg05575921-<em>AHRR</em> and cg01901332-<em>ARRB1</em>), there was limited evidence for heterogeneity in MR effect estimates (Q-test, P&gt;0.05, Supplementary Table 6).</p>
<p>Single mQTLs for cg05575921-<em>AHRR</em>, cg27241845-<em>ALPPL2</em>, and cg26963277-<em>KCNQ1</em> showed some evidence of association with smoking cessation (former vs. current smokers), although these associations were not below the FDR&lt;0.05 threshold (Supplementary Figure 4).</p>
<div id="potential-causal-effect-of-ahrr-methylation-on-lung-cancer-risk-one-sample-mr" class="section level4" number="7.4.2.1">
<h4><span class="header-section-number">7.4.2.1</span> Potential causal effect of <em>AHRR</em> methylation on lung cancer risk: one sample MR</h4>
<p>In the CCHS, a per (average methylation-increasing) allele change in a four-mQTL allele score was associated with a 0.73% [95% CI=0.56,0.90] increase in methylation (P&lt;1 x 10-10) and explained 0.8% of the variance in cg05575921-<em>AHRR</em> methylation (F-statistic=74.2). Confounding factors were not strongly associated with the genotypes in this cohort (P&gt;=0.11) (Supplementary Table 7). Results provided some evidence for an effect of cg05575921 methylation on total lung cancer risk (HR=0.30 [95% CI=0.10,1.00] per SD (9.2%) increase) (Supplementary Table 8). The effect estimate did not change substantively when stratified by smoking status (Supplementary Table 8).</p>
<p>Given contrasting findings with the main MR analysis, where cg05575921-<em>AHRR</em> methylation was not causally implicated in lung cancer, and the lower power in the one-sample analysis to detect an effect of equivalent size to the observational results (power = 19% at alpha = 0.05), we performed further two-sample MR based on the four mQTLs using data from both CCHS (sample one) and the TRICL-ILCCO consortium (sample two). Results showed no strong evidence for a causal effect of DNA methylation on total lung cancer risk (OR=1.00 [95% CI=0.83,1.10] per SD increase) (Supplementary Figure 5). There was also limited evidence for an effect of cg05575921-<em>AHRR</em> methylation when stratified by cancer subtype and smoking status (Supplementary Figure 5) and no strong evidence for heterogeneity of the mQTL effects (Supplementary Table 9). Conclusions were consistent when MR-Egger (27) was applied (Supplementary Figure 5) and when accounting for correlation structure between the mQTLs (Supplementary Table 9).</p>
</div>
</div>
<div id="tumour-and-adjacent-normal-lung-tissue-methylation-patterns" class="section level3" number="7.4.3">
<h3><span class="header-section-number">7.4.3</span> Tumour and adjacent normal lung tissue methylation patterns</h3>
<p>For cg05575921-<em>AHRR</em>, there was no strong evidence for differential methylation between adenocarcinoma tissue and adjacent healthy tissue (P=0.963), and weak evidence for hypermethylation in squamous cell carcinoma tissue (P=0.035) (Figure 4, Supplementary Table 10). For the other CpG sites there was evidence for a difference in DNA methylation between tumour and healthy adjacent tissue at several sites in both adenocarcinoma and squamous cell carcinoma, with consistent differences for CpG sites in ALPPL2 (cg2156642, cg05951221 and cg01940273), as well as cg23771366-PRSS23, cg26963277-KCNQ1, cg09935388-GFI1, cg0101332-ARRB1, cg08709672-AVPR1B and cg25305703-CASC21. However, hypermethylation in tumour tissue was found for the majority of these sites, which is opposite to what was observed in the EWAS analysis.</p>
<p>(Figure 4 here)</p>
</div>
<div id="gene-expression-associated-with-mqtls-in-blood-and-lung-tissue" class="section level3" number="7.4.4">
<h3><span class="header-section-number">7.4.4</span> Gene expression associated with mQTLs in blood and lung tissue</h3>
<p>Of the 10 genes annotated to the 14 CpG sites, eight genes were expressed sufficiently to be detected in lung (AVPR1B and CASC21 were not) and seven in blood (AVPR1B, CASC21 and ALPPL2 were not). Of these, gene expression of ARRB1 could not be investigated as the mQTLs in that region were not present in the GTEx data. rs3748971 and rs878481, mQTLs for cg21566642 and cg05951221 respectively, were associated with increased expression of ALPPL2 (P=0.002 and P=0.0001). No other mQTLs were associated with expression of the annotated gene at a Bonferroni corrected P value threshold (P&lt;0.05/19=0.0026) (Supplementary Table 11).</p>
</div>
</div>
<div id="discussion-1" class="section level2" number="7.5">
<h2><span class="header-section-number">7.5</span> Discussion</h2>
<p>In this study, we identified 16 CpG sites associated with lung cancer, of which 14 have been previously identified in relation to smoke exposure <span class="citation">(<span class="citeproc-not-found" data-reference-id="Joehanes2016"><strong>???</strong></span>)</span> 9 and six were highlighted in a previous study as being associated with lung cancer <span class="citation">(<span class="citeproc-not-found" data-reference-id="Baglietto2017"><strong>???</strong></span>)</span> 3. This previous study used the same data from the four cohorts investigated here, but in a discovery and replication, rather than meta-analysis framework. Overall, using MR we found limited evidence supporting a potential causal effect of methylation at the CpG sites identified in peripheral blood on lung cancer. These findings are in contrast to previous analyses suggesting that methylation at two CpG sites investigated (in <em>AHRR</em> and <em>F2RL3</em>) mediated &gt; 30% of the effect of smoking on lung cancer risk <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fasanelli2015"><strong>???</strong></span>)</span> 2. This previous study used methods which are sensitive to residual confounding and measurement error that may have biased results [<span class="citation">(<span class="citeproc-not-found" data-reference-id="Richmond2016"><strong>???</strong></span>)</span>; Hemani2017] 12, 35. These limitations are largely overcome using MR <span class="citation">(<span class="citeproc-not-found" data-reference-id="Richmond2016"><strong>???</strong></span>)</span> 12. While there was some evidence for an effect of methylation at some of the other CpG sites on risk of subtypes of lung cancer, these effects were not robust to multiple testing correction and were not validated in the analysis of tumour and adjacent normal lung tissue methylation nor in gene expression analysis.</p>
<p>A major strength of the study was the use of two-sample MR to integrate an extensive epigenetic resource and summary data from a large lung cancer GWAS to appraise causality of observational associations with &gt;99% power. Evidence against the observational findings were also acquired through tissue-specific DNA methylation and gene expression analyses.</p>
<p>Limitations include potential “winner’s curse” which may bias causal estimates in a two-sample MR analysis towards the null if the discovery sample for identifying genetic instruments is used as the first sample, as was done for our main MR analysis using data from ARIES <span class="citation">(<span class="citeproc-not-found" data-reference-id="Burgess2011"><strong>???</strong></span>)</span> 36. However, findings were similar when using replicated mQTLs in NSHDS, indicating the potential impact of this bias was minimal (Supplementary Figure 1). Another limitation relates to the potential issue of consistency and validity of the instruments across the two samples. For a minority of the mQTL-CpG associations (4 out of 16), there was limited replication across time points and in particular, 6 mQTLs were not strongly associated with DNA methylation in adults. Further, our primary data used for the first sample in the two-sample MR was ARIES, which contains no male adults. If the mQTLs identified vary by sex and time, then this could bias our results. However, our replication cohort NSHDS contains adult males. Therefore, the 10 mQTLs that replicated in NSHDS are unlikely to be biased by the sex discordance. Also, we replicated the findings for cg05575921 <em>AHRR</em> in CCHS, which contains both adult males and females, in a two-sample MR analysis, suggesting these results are also not influenced by sex discordance. Caution is therefore warranted when interpreting the null results for the two-sample MR estimates for the CpG sites for which mQTLs were not repliacted, which could be the result of weak-instrument bias.</p>
<p>The lack of independent mQTLs for each CpG site did not allow us to properly appraise horizontal pleiotropy in our MR analyses. Where possible we only included cis-acting mQTLs to minimise pleiotropy and investigated heterogeneity where there were multiple independent mQTLs. Three mQTLs were nominally associated with smoking phenotypes, but not to the extent that this would bias our MR results substantially. Some of the mQTLs used influence multiple CpGs in the same region, suggesting genomic control of methylation at a regional rather than single CpG level. This was untested, but methods to detect differentially methylated regions (DMRs) and identify genetic variants which proxy for them may be fruitful in probing the effect of methylation across gene regions.</p>
<p>A further limitation relates to the inconsistency in effect estimates between the one- and two-sample MR analysis to appraise the causal role of <em>AHRR</em> methylation. While findings in CCHS were supportive of a causal effect of <em>AHRR</em> methylation on lung cancer (HR=0.30 [95% CI=0.10,1.00] per SD), in two-sample MR this site was not causally implicated (OR=1.00 [95% CI=0.83,1.10] per SD increase). We verified that this was not due to differences in the genetic instruments used, nor due to issues of weak instrument bias. Given the CCHS one-sample MR had little power (19% at alpha = 0.05) to detect a causal effect with a size equivalent to that of the observational analysis, we have more confidence in the results from the two-sample approach.</p>
<p>Peripheral blood may not be the ideal tissue to assess the association between DNA methylation and lung cancer. While a high degree of concordance in mQTLs has been observed across lung tissue, skin and peripheral blood DNA <span class="citation">(<span class="citeproc-not-found" data-reference-id="Shi2014"><strong>???</strong></span>)</span> 37, we were unable to directly evaluate this here. A possible explanation for a lack of causal effect at <em>AHRR</em> is due to the limitation of tissue specificity as we found that the mQTLs used to instrument cg05575921 were not strongly related to expression of <em>AHRR</em> in lung tissue. However, findings from MR analysis were corroborated by the lack of evidence for differential methylation at <em>AHRR</em> between lung adenocarcinoma tissue and adjacent healthy tissue, and weak evidence for hypermethylation (opposite to the expected direction) in squamous cell lung cancer tissue. This result may be interesting in itself as smoking is hypothesized to influence squamous cell carcinoma more than adenocarcinoma. However, the result conflicts with that found in the MR analysis. Furthermore, another study investigating tumorous lung tissue (N=511) found only weak evidence for an association between smoking and cg05575921 <em>AHRR</em> methylation, that did not survive multiple testing correction (P=0.02) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Freeman2016"><strong>???</strong></span>)</span> 38. However, our results do not fully exclude <em>AHRR</em> from involvement in the disease process. <em>AHRR</em> and <em>AHR</em> form a regulatory feedback loop, which means that the actual effect of differential methylation or differential expression of <em>AHR</em>/<em>AHRR</em> on pathway activity is complex <span class="citation">(<span class="citeproc-not-found" data-reference-id="Chen2017"><strong>???</strong></span>)</span> 39. In addition, some of the CpG sites identified in the EWAS were found to be differentially methylated in the tumour and adjacent normal lung tissue comparison. While this could represent a false negative result of the MR analysis, it is of interest that differential methylation in the tissue comparison analysis was typically in the opposite direction to that observed in the EWAS. Furthermore, while this method can be used to minimize confounding, it does not fully eliminate the possibility of bias due to reverse causation (whereby cancer induces changes in DNA methylation) or intra-individual confounding e.g. by gene expression. Therefore, it doesn’t give conclusive evidence that DNA methylation changes at these sites are not relevant to the development of lung cancer.</p>
<p>While DNA methylation in peripheral blood may be predictive of lung cancer risk, according to the present analysis it is unlikely to play a causal role in lung carcinogenesis at the CpG sites investigated. Findings from this study issue caution over the use of traditional mediation analyses to implicate intermediate biomarkers (such as DNA methylation) in pathways linking an exposure with disease, given the potential for residual confounding in this context <span class="citation">(<span class="citeproc-not-found" data-reference-id="Richmond2016"><strong>???</strong></span>)</span> 12. However, the findings of this study do not preclude the possibility that other DNA methylation changes are causally related to lung cancer (or other smoking-associated disease) <span class="citation">(<span class="citeproc-not-found" data-reference-id="Gao2016"><strong>???</strong></span>)</span> 40.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="6-ewas-gwas-comp-chapter.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
