<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Preface | thesis.utf8</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.18.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Preface | thesis.utf8" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Preface | thesis.utf8" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="1-introduction.html"/>
<script src="libs/header-attrs-2.1.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#dna-methylation-as-part-of-the-regulatory-machinery"><i class="fa fa-check"></i><b>1.1</b> DNA methylation as part of the regulatory machinery</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#gene-expression-regulation"><i class="fa fa-check"></i><b>1.1.1</b> Gene expression regulation</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#defining-epigenetics"><i class="fa fa-check"></i><b>1.1.2</b> Defining epigenetics</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#histone-modifications"><i class="fa fa-check"></i><b>1.1.3</b> Histone modifications</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-introduction.html"><a href="1-introduction.html#dna-methylation"><i class="fa fa-check"></i><b>1.1.4</b> DNA methylation</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#dnam-phs"><i class="fa fa-check"></i><b>1.2</b> Use of DNA methylation in a population setting </a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="1-introduction.html"><a href="1-introduction.html#appeal-of-dnam"><i class="fa fa-check"></i><b>1.2.1</b> The appeal of DNA methylation to epidemiologists</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-introduction.html"><a href="1-introduction.html#ewas"><i class="fa fa-check"></i><b>1.2.2</b> Epigenome-wide association studies</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-introduction.html"><a href="1-introduction.html#problems-for-ewas"><i class="fa fa-check"></i><b>1.2.3</b> Problems for EWAS</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#genetics-in-ewas"><i class="fa fa-check"></i><b>1.3</b> Using methods from genetics to help inform future EWAS</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#gwas-catalog"><i class="fa fa-check"></i><b>1.3.1</b> The GWAS Catalog</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#heritability"><i class="fa fa-check"></i><b>1.3.2</b> Total variance captured by all sites measured genome-wide</a></li>
<li class="chapter" data-level="1.3.3" data-path="1-introduction.html"><a href="1-introduction.html#inferring-biology-from-signals"><i class="fa fa-check"></i><b>1.3.3</b> Inferring biology from signals</a></li>
<li class="chapter" data-level="1.3.4" data-path="1-introduction.html"><a href="1-introduction.html#establishing-causality"><i class="fa fa-check"></i><b>1.3.4</b> Establishing causality</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#overview-of-thesis-aims"><i class="fa fa-check"></i><b>1.4</b> Overview of thesis aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-methods.html"><a href="2-methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a></li>
<li class="chapter" data-level="3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html"><i class="fa fa-check"></i><b>3</b> The EWAS Catalog: a database of epigenome-wide association studies</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#implementation"><i class="fa fa-check"></i><b>3.3.1</b> Implementation</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#overview-of-publication-data-extraction"><i class="fa fa-check"></i><b>3.3.2</b> Overview of publication data extraction</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#overview-of-geo-data-extraction"><i class="fa fa-check"></i><b>3.3.3</b> Overview of GEO data extraction</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#ewas-methods"><i class="fa fa-check"></i><b>3.3.4</b> EWAS methods</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#database-interface-and-use"><i class="fa fa-check"></i><b>3.4.1</b> Database interface and use</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-properties-of-ewas.html"><a href="4-properties-of-ewas.html"><i class="fa fa-check"></i><b>4</b> Properties of EWAS</a></li>
<li class="chapter" data-level="5" data-path="5-h2ewas-chapter.html"><a href="5-h2ewas-chapter.html"><i class="fa fa-check"></i><b>5</b> h2ewas</a></li>
<li class="chapter" data-level="6" data-path="6-ewas-gwas-comp-chapter.html"><a href="6-ewas-gwas-comp-chapter.html"><i class="fa fa-check"></i><b>6</b> EWAS-GWAS comparison</a></li>
<li class="chapter" data-level="7" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html"><i class="fa fa-check"></i><b>7</b> Appraising the causal relevance of DNA methylation for risk of lung cancer</a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#abstract-1"><i class="fa fa-check"></i><b>7.1</b> Abstract</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#background"><i class="fa fa-check"></i><b>7.1.1</b> Background</a></li>
<li class="chapter" data-level="7.1.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#methods-2"><i class="fa fa-check"></i><b>7.1.2</b> Methods</a></li>
<li class="chapter" data-level="7.1.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#results-1"><i class="fa fa-check"></i><b>7.1.3</b> Results</a></li>
<li class="chapter" data-level="7.1.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#conclusions"><i class="fa fa-check"></i><b>7.1.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#introduction-2"><i class="fa fa-check"></i><b>7.2</b> Introduction</a></li>
<li class="chapter" data-level="7.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#methods-3"><i class="fa fa-check"></i><b>7.3</b> Methods</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-study-details"><i class="fa fa-check"></i><b>7.3.1</b> EWAS study details</a></li>
<li class="chapter" data-level="7.3.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-meta-analysis"><i class="fa fa-check"></i><b>7.3.2</b> EWAS Meta-analysis</a></li>
<li class="chapter" data-level="7.3.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#mendelian-randomization-1"><i class="fa fa-check"></i><b>7.3.3</b> Mendelian randomization</a></li>
<li class="chapter" data-level="7.3.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#sample-2-transdisciplinary-research-in-cancer-of-the-lung-and-the-international-lung-cancer-consortium"><i class="fa fa-check"></i><b>7.3.4</b> Sample 2: Transdisciplinary Research in Cancer of the Lung and The International Lung Cancer Consortium</a></li>
<li class="chapter" data-level="7.3.5" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#supplementary-analyses"><i class="fa fa-check"></i><b>7.3.5</b> Supplementary analyses</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#results-2"><i class="fa fa-check"></i><b>7.4</b> Results</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-meta-analysis-1"><i class="fa fa-check"></i><b>7.4.1</b> EWAS meta-analysis</a></li>
<li class="chapter" data-level="7.4.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#mendelian-randomization-2"><i class="fa fa-check"></i><b>7.4.2</b> Mendelian randomization</a></li>
<li class="chapter" data-level="7.4.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#tumour-and-adjacent-normal-lung-tissue-methylation-patterns"><i class="fa fa-check"></i><b>7.4.3</b> Tumour and adjacent normal lung tissue methylation patterns</a></li>
<li class="chapter" data-level="7.4.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#gene-expression-associated-with-mqtls-in-blood-and-lung-tissue"><i class="fa fa-check"></i><b>7.4.4</b> Gene expression associated with mQTLs in blood and lung tissue</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#discussion-1"><i class="fa fa-check"></i><b>7.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="B" data-path="B-the-second-appendix-for-fun.html"><a href="B-the-second-appendix-for-fun.html"><i class="fa fa-check"></i><b>B</b> The Second Appendix, for Fun</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<!--bookdown:title:end-->
<!--bookdown:title:start-->
<div id="preface" class="section level1 unnumbered" number="">
<h1>Preface</h1>
<p>This template is based on (and in many places copied directly from) the Reed College LaTeX template, but hopefully it will provide a nicer interface for those that have never used TeX or LaTeX before. Using <em>R Markdown</em> will also allow you to easily keep track of your analyses in <strong>R</strong> chunks of code, with the resulting plots and output included as well. The hope is this <em>R Markdown</em> template gets you in the habit of doing reproducible research, which benefits you long-term as a researcher, but also will greatly help anyone that is trying to reproduce or build onto your results down the road.</p>
<p>Hopefully, you won’t have much of a learning period to go through and you will reap the benefits of a nicely formatted thesis. The use of LaTeX in combination with <em>Markdown</em> is more consistent than the output of a word processor, much less prone to corruption or crashing, and the resulting file is smaller than a Word file. While you may have never had problems using Word in the past, your thesis is likely going to be about twice as large and complex as anything you’ve written before, taxing Word’s capabilities. After working with <em>Markdown</em> and <strong>R</strong> together for a few weeks, we are confident this will be your reporting style of choice going forward.</p>
<!-- 
If you're still on the fence about using _R Markdown_, check out the resource for newbies available at <https://ismayc.github.io/rbasics-book/> or email us at <data@reed.edu>. 
-->
<p><strong>Why use it?</strong></p>
<p><em>R Markdown</em> creates a simple and straightforward way to interface with the beauty of LaTeX. Packages have been written in <strong>R</strong> to work directly with LaTeX to produce nicely formatting tables and paragraphs. In addition to creating a user friendly interface to LaTeX, <em>R Markdown</em> also allows you to read in your data, to analyze it and to visualize it using <strong>R</strong> functions, and also to provide the documentation and commentary on the results of your project. Further, it allows for <strong>R</strong> results to be passed inline to the commentary of your results. You’ll see more on this later.</p>
<!-- 
Having your code and commentary all together in one place has a plethora of benefits!
-->
<p><strong>Who should use it?</strong></p>
<p>Anyone who needs to use data analysis, math, tables, a lot of figures, complex cross-references, or who just cares about the final appearance of their document should use <em>R Markdown</em>. Of particular use should be anyone in the sciences, but the user-friendly nature of <em>Markdown</em> and its ability to keep track of and easily include figures, automatically generate a table of contents, index, references, table of figures, etc. should make it of great benefit to nearly anyone writing a thesis project.</p>
<p><strong>For additional help with bookdown</strong>
Please visit <a href="https://bookdown.org/yihui/bookdown/">the free online bookdown reference guide</a>.</p>

</div>
            </section>

          </div>
        </div>
      </div>

<a href="1-introduction.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
